{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "34d52f30",
   "metadata": {},
   "outputs": [],
   "source": [
    "import json, requests\n",
    "from bs4 import BeautifulSoup\n",
    "from datetime import datetime, timedelta\n",
    "\n",
    "import feedparser\n",
    "# from markdownify import markdownify"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 64,
   "id": "2bbc7743",
   "metadata": {},
   "outputs": [],
   "source": [
    "nature_url = \"https://www.nature.com\"\n",
    "subject_url = nature_url + \"/search?article_type=research, reviews, protocols&subject={}&date_range=last_7_days&order=date_desc\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 65,
   "id": "f3fe9d33",
   "metadata": {},
   "outputs": [],
   "source": [
    "subjects = [\"Physics\", \"Chemistry\", \"Engineering\", \"Materials science\", \n",
    "    \"Energy science and technology\", \"Mathematics and computing\", \"Astronomy and planetary science\", \n",
    "    \"Business and industry\", \"Energy and society\", \"Geography\", \"Science\", \"technology and society\", \n",
    "    \"Social policy\", \"Politics and international relations\",\"Computational biology and bioinformatics\", \n",
    "    \"Chemical biology, Biotechnology\", \"Information systems and information technology\", \"Computer science\", \n",
    "    \"Machine learning\", \"Image processing\", \"Signal processing\"]\n",
    "\n",
    "subjects = [i.lower().replace(\" \",\"-\") for i in subjects]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 42,
   "id": "a791f536",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "datetime.datetime(2023, 3, 20, 0, 0)"
      ]
     },
     "execution_count": 42,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "date_str = \"20 March 2023\"\n",
    "date_obj = datetime.strptime(date_str, '%d %B %Y')\n",
    "date_obj"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 44,
   "id": "08430961",
   "metadata": {},
   "outputs": [],
   "source": [
    "r = requests.request(\"GET\", subject_url.format(\"biochemistry\"))\n",
    "soup = BeautifulSoup(r.content, \"html.parser\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 53,
   "id": "f07f4579",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'/articles/s41467-023-37281-7'"
      ]
     },
     "execution_count": 53,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "soup.find(\"a\",{\"class\":\"c-card__link u-link-inherit\"})[\"href\"]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 60,
   "id": "27e55f45",
   "metadata": {},
   "outputs": [],
   "source": [
    "url = nature_url+soup.find(\"a\",{\"class\":\"c-card__link u-link-inherit\"})[\"href\"]\n",
    "r = requests.request(\"GET\", url)\n",
    "soup = BeautifulSoup(r.content, \"html.parser\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 63,
   "id": "03b4dc8e",
   "metadata": {},
   "outputs": [],
   "source": [
    "soup.find(\"div\", {\"class\": \"c-article-body main-content\"})"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 75,
   "id": "a106ad32",
   "metadata": {},
   "outputs": [],
   "source": [
    "def news_content(url: str) -> str:\n",
    "    \n",
    "    r = requests.request(\"GET\", url)\n",
    "    soup = BeautifulSoup(r.content, \"html.parser\")\n",
    "    \n",
    "    if soup.find(\"div\", {\"class\": \"c-article-body main-content\"}):\n",
    "        \n",
    "        field = ','.join([i.lower() for i in soup.find(\"ul\",{\"class\":\"c-article-subjects u-clearfix u-hide-print\"}).text.strip().split(\"\\n\\n\\n\")])\n",
    "        \n",
    "        html_content = soup.find(\"div\",{\"class\":\"c-article-body main-content\"})\n",
    "        soup.find(\"figcaption\").decompose()\n",
    "        \n",
    "        content = html_content.text\n",
    "        img_link = \"https:\" + soup.find(\"figure\",{\"class\":\"figure\"}).img[\"src\"]\n",
    "        \n",
    "        return field, content, img_link\n",
    "        \n",
    "    return None"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 76,
   "id": "08e0e43c",
   "metadata": {},
   "outputs": [],
   "source": [
    "def one_week_articles() -> list:\n",
    "\n",
    "    articles = list()\n",
    "    \n",
    "    nature_rss = feedparser.parse(\"https://www.nature.com/nature.rss\")\n",
    "    \n",
    "    for item in nature_rss[\"entries\"]:\n",
    "        \n",
    "        try:\n",
    "            \n",
    "            article_link = item[\"id\"]\n",
    "        \n",
    "            article_date = datetime.strptime(item[\"updated\"], '%Y-%m-%d') \n",
    "\n",
    "            if (datetime.today() - timedelta(weeks=1)) < article_date < datetime.today():\n",
    "\n",
    "                _data = news_content(article_link)\n",
    "                field, _content, img_link = _data\n",
    "\n",
    "                if _data is not None:\n",
    "\n",
    "                    article = {}\n",
    "\n",
    "                    article[\"title\"] = item[\"title\"]\n",
    "                    article[\"summary\"] = item[\"summary\"]\n",
    "\n",
    "                    article[\"field\"] = field\n",
    "                    article[\"content\"] = _content\n",
    "                    article[\"img_link\"] = img_link\n",
    "\n",
    "                    articles.append(article)\n",
    "        except Exception as Error:\n",
    "            \n",
    "            print(f\"Cant not scarpe { article_link } due to - {Error}\")\n",
    "\n",
    "    return articles"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 77,
   "id": "f1bdac12",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Cant not scarpe https://www.nature.com/articles/d41586-023-00844-1 due to - cannot unpack non-iterable NoneType object\n",
      "Cant not scarpe https://www.nature.com/articles/d41586-023-00845-0 due to - cannot unpack non-iterable NoneType object\n",
      "Cant not scarpe https://www.nature.com/articles/d41586-023-00843-2 due to - cannot unpack non-iterable NoneType object\n",
      "Cant not scarpe https://www.nature.com/articles/d41586-023-00708-8 due to - cannot unpack non-iterable NoneType object\n",
      "Cant not scarpe https://www.nature.com/articles/s41586-023-05950-8 due to - cannot unpack non-iterable NoneType object\n",
      "Cant not scarpe https://www.nature.com/articles/s41586-023-05958-0 due to - cannot unpack non-iterable NoneType object\n",
      "Cant not scarpe https://www.nature.com/articles/d41586-023-00799-3 due to - cannot unpack non-iterable NoneType object\n",
      "Cant not scarpe https://www.nature.com/articles/d41586-023-00853-0 due to - 'NoneType' object has no attribute 'text'\n",
      "Cant not scarpe https://www.nature.com/articles/d41586-023-00776-w due to - cannot unpack non-iterable NoneType object\n",
      "Cant not scarpe https://www.nature.com/articles/d41586-023-00774-y due to - cannot unpack non-iterable NoneType object\n",
      "Cant not scarpe https://www.nature.com/articles/d41586-023-00839-y due to - 'NoneType' object has no attribute 'text'\n",
      "Cant not scarpe https://www.nature.com/articles/d41586-023-00742-6 due to - 'NoneType' object has no attribute 'text'\n",
      "Cant not scarpe https://www.nature.com/articles/s41586-023-05932-w due to - cannot unpack non-iterable NoneType object\n",
      "Cant not scarpe https://www.nature.com/articles/d41586-023-00775-x due to - cannot unpack non-iterable NoneType object\n",
      "Cant not scarpe https://www.nature.com/articles/s41586-023-05799-x due to - cannot unpack non-iterable NoneType object\n",
      "Cant not scarpe https://www.nature.com/articles/d41586-023-00762-2 due to - 'NoneType' object has no attribute 'text'\n",
      "Cant not scarpe https://www.nature.com/articles/d41586-023-00829-0 due to - 'NoneType' object has no attribute 'text'\n",
      "Cant not scarpe https://www.nature.com/articles/s41586-023-05929-5 due to - cannot unpack non-iterable NoneType object\n",
      "Cant not scarpe https://www.nature.com/articles/s41586-023-05756-8 due to - cannot unpack non-iterable NoneType object\n",
      "Cant not scarpe https://www.nature.com/articles/s41586-023-05939-3 due to - cannot unpack non-iterable NoneType object\n",
      "Cant not scarpe https://www.nature.com/articles/d41586-023-00785-9 due to - 'NoneType' object has no attribute 'decompose'\n",
      "Cant not scarpe https://www.nature.com/articles/s41586-023-05719-z due to - cannot unpack non-iterable NoneType object\n",
      "Cant not scarpe https://www.nature.com/articles/s41586-023-05757-7 due to - cannot unpack non-iterable NoneType object\n",
      "Cant not scarpe https://www.nature.com/articles/s41586-023-05798-y due to - cannot unpack non-iterable NoneType object\n",
      "Cant not scarpe https://www.nature.com/articles/s41586-023-05812-3 due to - cannot unpack non-iterable NoneType object\n",
      "Cant not scarpe https://www.nature.com/articles/s41586-023-05831-0 due to - cannot unpack non-iterable NoneType object\n",
      "Cant not scarpe https://www.nature.com/articles/s41586-023-05800-7 due to - cannot unpack non-iterable NoneType object\n",
      "Cant not scarpe https://www.nature.com/articles/s41586-023-05824-z due to - cannot unpack non-iterable NoneType object\n",
      "Cant not scarpe https://www.nature.com/articles/s41586-023-05794-2 due to - cannot unpack non-iterable NoneType object\n",
      "Cant not scarpe https://www.nature.com/articles/s41586-023-05834-x due to - cannot unpack non-iterable NoneType object\n",
      "Cant not scarpe https://www.nature.com/articles/s41586-023-05733-1 due to - cannot unpack non-iterable NoneType object\n",
      "Cant not scarpe https://www.nature.com/articles/s41586-023-05801-6 due to - cannot unpack non-iterable NoneType object\n",
      "Cant not scarpe https://www.nature.com/articles/s41586-023-05721-5 due to - cannot unpack non-iterable NoneType object\n",
      "Cant not scarpe https://www.nature.com/articles/s41586-023-05792-4 due to - cannot unpack non-iterable NoneType object\n",
      "Cant not scarpe https://www.nature.com/articles/s41586-023-05724-2 due to - cannot unpack non-iterable NoneType object\n",
      "Cant not scarpe https://www.nature.com/articles/s41586-022-05679-w due to - cannot unpack non-iterable NoneType object\n",
      "Cant not scarpe https://www.nature.com/articles/s41586-023-05795-1 due to - cannot unpack non-iterable NoneType object\n",
      "Cant not scarpe https://www.nature.com/articles/s41586-022-05686-x due to - cannot unpack non-iterable NoneType object\n",
      "Cant not scarpe https://www.nature.com/articles/s41586-022-05665-2 due to - cannot unpack non-iterable NoneType object\n",
      "Cant not scarpe https://www.nature.com/articles/s41586-023-05790-6 due to - cannot unpack non-iterable NoneType object\n",
      "Cant not scarpe https://www.nature.com/articles/s41586-023-05828-9 due to - cannot unpack non-iterable NoneType object\n",
      "Cant not scarpe https://www.nature.com/articles/s41586-023-05755-9 due to - cannot unpack non-iterable NoneType object\n",
      "Cant not scarpe https://www.nature.com/articles/s41586-023-05793-3 due to - cannot unpack non-iterable NoneType object\n",
      "Cant not scarpe https://www.nature.com/articles/d41586-023-00698-7 due to - 'NoneType' object has no attribute 'text'\n",
      "Cant not scarpe https://www.nature.com/articles/d41586-023-00439-w due to - cannot unpack non-iterable NoneType object\n",
      "Cant not scarpe https://www.nature.com/articles/d41586-023-00705-x due to - cannot unpack non-iterable NoneType object\n",
      "Cant not scarpe https://www.nature.com/articles/d41586-023-00755-1 due to - cannot unpack non-iterable NoneType object\n",
      "Cant not scarpe https://www.nature.com/articles/d41586-023-00317-5 due to - cannot unpack non-iterable NoneType object\n",
      "Cant not scarpe https://www.nature.com/articles/d41586-023-00746-2 due to - 'NoneType' object has no attribute 'text'\n",
      "Cant not scarpe https://www.nature.com/articles/d41586-023-00814-7 due to - 'NoneType' object has no attribute 'text'\n",
      "Cant not scarpe https://www.nature.com/articles/d41586-023-00706-w due to - cannot unpack non-iterable NoneType object\n",
      "Cant not scarpe https://www.nature.com/articles/d41586-023-00426-1 due to - cannot unpack non-iterable NoneType object\n",
      "Cant not scarpe https://www.nature.com/articles/d41586-023-00702-0 due to - cannot unpack non-iterable NoneType object\n",
      "Cant not scarpe https://www.nature.com/articles/d41586-023-00424-3 due to - cannot unpack non-iterable NoneType object\n"
     ]
    }
   ],
   "source": [
    "data = one_week_articles()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 79,
   "id": "8ac7d57d",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "21"
      ]
     },
     "execution_count": 79,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(data)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 80,
   "id": "c6bc78c6",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[{'title': 'How not to chatter like a toddler when giving a scientific presentation',\n",
       "  'summary': '<p>Nature, Published online: 21 March 2023; <a href=\"https://www.nature.com/articles/d41586-023-00832-5\">doi:10.1038/d41586-023-00832-5</a></p>Are we hardwired to overstuff presentations with details? Four simple steps can overcome this tendency, says David Rubenson.',\n",
       "  'field': 'careers,conferences and meetings,communication',\n",
       "  'content': '\\n\\n\\n\\n\\n\\n\\nI have a new mentor: my two-and-a-half-year-old grandson. Like many toddlers, he rambles continuously, with little concern as to whether anyone understands him. It struck me that, if a toddler uses language in this way, it’s no surprise that scientific presentations — which I focus on in my job as a presentation coach — can themselves be overly detailed and undecipherable. A toddler’s loquaciousness creeps into many scientific talks — perhaps it’s an instinct we don’t completely outgrow.This might explain why making scientists aware of the problem isn’t enough to correct it. Many complain about excessive detail in colleagues’ presentations but fail to address the issue in their own. The desire to insert one more overly detailed slide seems irresistible. Presenters need specific steps that limit the scope of their talks and eliminate the instinct to ramble on.Based on 20 years of experience teaching presentation techniques at many research institutions, I’ve developed some strategies to disrupt this tendency. These four steps will help scientists to create clearer, more-concise presentations.Write a simple narrative firstBefore creating slides, write down the main points you hope to communicate. These might include an overarching question, the reason that question is important, your key results and any remaining issues. Try to connect the points into a smooth half-page narrative with a beginning, a middle and an end.Your narrative is the foundation on which you can develop talks of varying lengths. Longer talks might contain more supporting data and detail, but still need to be based on a few key points. Audiences are much more likely to ‘lose the flow’ during a lengthy, data-rich talk than a short one.A tip for those who struggle with writing: you could use an abstract for a journal article as your starting point, then try to make it more conversational, with a continuous flow.Whether you have a journal article or not, realize that the audience will remember few, if any, specific slides the day after your talk. So, when constructing your narrative, think about the two or three main ideas you hope they will remember. These can constitute the centrepiece of the narrative, supplemented with appropriate introductory and concluding material.Practise with two types of listenerFirst, try out your narrative, without the slides, on someone who works in your field. They might identify issues that have not occurred to you.Then think about the breadth of your target audience and try the narrative out on a scientist in that audience whose speciality has the least amount of overlap with your own.If these listeners express confusion, that is better than if they are polite and keep quiet. Don’t respond by chattering like a toddler. Instead, prod them to articulate what they don’t understand. It’s easier, more efficient and less intimidating to revise the organization of your talk before designing slides.Build up, don’t cut downUse your narrative to create a 5-minute presentation, with slides. Then ask, what extra slides would comprise a 10-minute session? Keep adding material in small increments until you reach the actual time allocated for your talk. Always use your narrative as the basis for deciding what to include and, more importantly, what not to include.Build up your slides, tooEach slide should have an overarching message that explains an aspect of the narrative. The slide title should be that message or consist of words that help to convey it.Work out the message and title first. Then include the minimum amount of information or data needed to validate the message. Consider adding detail if it strengthens the message and the slide remains comprehensible, but err on the side of simplicity. Audience members who want to know more can contact you subsequently.For example, the initial draft of a slide could show a bar chart of numbers of clinical trials in each of the past ten years. After examining it, you might feel that the slide is simple enough to divide each bar into categories such as ‘cancer’, ‘heart disease’ and ‘stroke’. If not, then a later set of slides could show more-detailed information.You might find it useful to sketch slides by hand initially. This helps to maintain focus on each slide’s overall purpose, rather than allowing yourself to be diverted by font sizes and other layout details, especially if you aren’t fluent in using presentation software. Coming up with purposeful slides is the top priority.\\n',\n",
       "  'img_link': 'https://media.nature.com/lw767/magazine-assets/d41586-023-00832-5/d41586-023-00832-5_24090966.jpg'},\n",
       " {'title': 'I work in an ultra-clean room on ultra-small chips',\n",
       "  'summary': '<p>Nature, Published online: 20 March 2023; <a href=\"https://www.nature.com/articles/d41586-023-00795-7\">doi:10.1038/d41586-023-00795-7</a></p>Luis Fonseca leads research into micro-, nano- and photonic chip technologies that could kill cancer cells, protect the environment and facilitate space research.',\n",
       "  'field': 'careers,physics,nanoscience and technology',\n",
       "  'content': '\\nThe Institute of Microelectronics of Barcelona, in Spain, where we research and develop micro- and nanotechnology, hosts the largest ‘clean room’ in southern Europe, at 1,500 square metres. A clean room is a controlled environment where pollutants such as dust, airborne microorganisms and aerosol particles are filtered out.The picture above was taken in the clean room’s photolithographic laboratory. This is where we transform silicon wafers into micro-, nano- or photonic chips. We use yellow light in here because the fabrication process is sensitive to the ultraviolet light in standard lighting.The air in this part of the clean room is 10,000 times cleaner than ‘normal’ air. The air filters block any particle that could interfere with the fabrication. They also change all of the air in the room about once a minute.We wear protective suits like the one I have on in the picture to protect the chips; unwanted particles could make them defective.Here, I’m looking through the microscope of a contact-alignment instrument. This ensures the correct alignment of the layers of silicon dioxide and other metals that are deposited on the wafers as part of our chip-manufacturing process.Including technicians and engineers, the clean room has a team of around 40 people. We research how our technologies can be applied in many different fields: health, the environment, energy and space exploration.We have developed a way to make chips tiny enough to enter living cells. These nanostructures allow scientists to identify and track living cells and to understand how these cells work from within.The tiny chips can also be used as nanotools to keep cancer cells from developing. In this way, we hope to find alternatives to chemotherapy and other treatments that are too aggressive for the human body.\\n',\n",
       "  'img_link': 'https://media.nature.com/w767/magazine-assets/d41586-023-00795-7/d41586-023-00795-7_24630594.jpg'},\n",
       " {'title': 'Paul Berg (1926–2023)',\n",
       "  'summary': '<p>Nature, Published online: 20 March 2023; <a href=\"https://www.nature.com/articles/d41586-023-00792-w\">doi:10.1038/d41586-023-00792-w</a></p>Biochemist who invented recombinant DNA technology.',\n",
       "  'field': 'history,biotechnology,dna sequencing,genetics,genomics',\n",
       "  'content': '\\n\\n\\n\\n\\n\\n\\nPaul Berg was the first researcher to incorporate DNA from one species into the genetic material of another. In so doing, he invented one of the most powerful tools in modern biology and the basis for a biotechnology industry that is worth hundreds of billions of dollars and has designed therapies for many human diseases. This alone would have been enough to establish Berg’s pre-eminent status — he received the Nobel Prize in Chemistry in 1980 — but he is also remembered for addressing anxieties about the possible risks of ‘recombinant’ research, and leading the development of a consensus on responsible use of the technology. He died on 15 February, aged 96.Berg grew up and was educated in New York City and attended Abraham Lincoln High School in Brooklyn. This was the alma mater of another Nobel laureate, biochemist Arthur Kornberg, who later became a long-standing mentor to Berg. Berg’s undergraduate education in biochemistry at the Pennsylvania State University in University Park was interrupted by the outbreak of the Second World War, during which he served in the US Navy. To his disappointment, his passionate wish to become a naval pilot was never realized.Berg went on to study biochemistry at what is now Case Western Reserve University in Cleveland, Ohio, in 1948. After a postdoc with biochemist Herman Kalckar in Copenhagen, Berg moved to the Department of Microbiology at Washington University in St. Louis, Missouri, to work with Kornberg. There, he gained international attention for reporting the role of a new intermediate molecule in the metabolism of fatty acids. Having gained faculty status, he went on to discover transfer RNA — first isolated by others in cell-free liver extracts — in extracts of Escherichia coli, and showed that amino acids are activated before being attached to transfer RNA for assembly into proteins.\\n\\nAn interview with Paul Berg\\nIn 1959, Berg relocated with Kornberg to Stanford University in California. Becoming interested in exploring gene function further, he spent a sabbatical year at the Salk Institute in San Diego, California, with virologist and cancer biologist Renato Dulbecco. This led Berg to retool his Stanford laboratory to work with animal cells that serve as hosts for polyoma viruses, such as the potentially cancer-causing DNA virus SV40. He pursued techniques that would allow him to study the expression of viral genes in prokaryotes such as E. coli, and the expression of prokaryotic genes in animal cells.To do this, he devised a laborious and tricky strategy for generating DNA molecules that contain DNA from more than one species — recombinant DNAs. This involved using enzymes to snip open the circular genome of the SV40 virus and to snip out a fragment containing three genes from the DNA of the bacterium E. coli, then tailoring the free ends of both molecules so that they would rejoin as a single circle of DNA.The idea was to use the modified SV40 as a viral vector to carry ‘foreign’ genetic material into animal cells. But soon after he initiated his experiments, concerns arose that introducing SV40 genes into E. coli and other common human intestinal bacteria could raise the risk of human cancer. This concern prompted Berg to call a moratorium on all recombinant DNA work in his laboratory until the issue could be explored more thoroughly.As word spread about what Berg was attempting, elaborate theoretical catastrophic scenarios emerged, including the possible accidental generation of Frankenstein-like monsters — and even interference with natural evolution. Berg and his colleagues responded by convening two meetings at the Asilomar conference centre just south of San Francisco, California. The first, held in 1973, prompted Berg to circulate what became known as the Berg Letter to several journals, including Nature, requesting that members of the scientific community temporarily cease experiments involving recombinant DNA. The historic second meeting was held in 1975 to debate the pros and cons of recombinant DNA technology.\\n\\nArthur Kornberg (1918–2007)\\nConscious of the wide public interest, Berg and the other organizers made a point of inviting members of the press to the 1975 meeting, and encouraged them to ask questions, but requested they not file their reports until the conference had concluded. The conference participants agreed to a voluntary moratorium on research until guidelines could be promulgated for the safe use of the new technology. The US National Institutes of Health first published its guidelines in 1976, and these have been regularly updated ever since. As a result, the very real threat of a total ban on recombinant DNA research was averted, opening the door to genome sequencing and editing, and making possible countless advances in basic research, clinical medicine and drug discovery, and other applications of biotechnology.Berg devoted huge amounts of time to promoting the safe use of recombinant DNA technology, which he went on to deploy towards clinical application. In 1980, with Kornberg and US geneticist Charles Yanofsky, he founded the DNAX Research Institute of Cellular and Molecular Biology in Palo Alto, California. He also served as the first director of Stanford’s multidisciplinary Beckman Center for Molecular and Genetic Medicine, opened in 1985.Berg was an extraordinary scientist, teacher, mentor and administrator, and a skilled proponent of bringing good, old-fashioned common sense to bear on contentious public-policy issues, including later the use of embryonic stem cells in biomedical research. He formally closed his research laboratory at the turn of the twenty-first century, but continued to advise his department and his university, and to act as a consultant to biotechnology companies. The world of academia will sorely miss this giant of the early days of molecular biology and molecular genetics.\\n',\n",
       "  'img_link': 'https://media.nature.com/lw767/magazine-assets/d41586-023-00792-w/d41586-023-00792-w_24085710.jpg'},\n",
       " {'title': 'Ohio train derailment: scientists scan for lingering toxics',\n",
       "  'summary': '<p>Nature, Published online: 20 March 2023; <a href=\"https://www.nature.com/articles/d41586-023-00820-9\">doi:10.1038/d41586-023-00820-9</a></p>East Palestine residents look to independent researchers to fill gaps left by authorities.',\n",
       "  'field': 'environmental sciences,chemistry,public health,scientific community',\n",
       "  'content': '\\n\\n\\n\\n\\n\\n\\nResidents of East Palestine, Ohio, have complained of symptoms including headaches and difficulty breathing following a nearby train derailment that spilled millions of litres of industrial chemicals last month. Officials at the US Environmental Protection Agency (EPA) and local, state and federal agencies, as well as the train’s operator, Norfolk Southern, have been monitoring water and air quality in the town — but community members haven’t been satisfied with official reports claiming that levels of the chemicals are low and safe.\\n\\nHow a dangerous stew of air pollution is choking the United States\\nSome independent researchers have arrived on the scene, hoping to help with more measurements and analyses. Others have been invited by residents who want a second opinion. The teams think they can help to fill the gaps left by authorities.“The residents had a disconnect between them experiencing some symptoms, and everybody telling them everything was okay,” says Ivan Rusyn, director of the Texas A&M University Superfund Research Center in College Station.Rusyn is part of a group of researchers at Texas A&M and Carnegie Mellon University in Pittsburgh, Pennsylvania, that drove a mobile laboratory around East Palestine on 20 and 21 February to measure air quality. Many of their measurements agreed with the EPA’s. But the researchers did detect levels of acrolein — a chemical irritant that affects the eyes, skin and respiratory system — that were three times as high as those in downtown Pittsburgh, some 80 kilometres away. (On a normal day, rural areas such as East Palestine usually have lower levels of air pollutants than do urban ones.) The scientists say that, if this level persists, it could affect residents’ health.On 6 February, three days after the train accident, authorities evacuated a 5-square-kilometre area that includes East Palestine. They punctured the train cars to drain the chemicals they had been carrying, including vinyl chloride and butyl acrylate, into pits. This was followed by a ‘controlled burn’ of the materials, to avoid an explosion, which generated an enormous black cloud over the site and probably created acrolein and other combustion by-products.In a statement to Nature, the EPA said that it had detected slightly elevated levels of acrolein in East Palestine in the weeks following the derailment, but that these levels do not pose health risks in the short term. “Those concentrations have since returned to levels below the national median,” the agency said.More sampling neededThe Texas A&M–Carnegie Mellon team’s findings highlight the importance of looking at a broader suite of chemicals than just those spilled during the train derailment, or ones that were likely to have formed during the controlled burn. Doing a ‘non-targeted analysis’ to find less-obvious compounds takes some time but is important, Albert Presto, a member of Carnegie Mellon’s Center for Atmospheric Particle Studies, said during an online briefing on 3 March.\\n\\nTainted water: the scientists tracing thousands of fluorinated chemicals in our environment\\nIn addition to acrolein, the group found higher concentrations of four more acrolein-like compounds around East Palestine than in surrounding areas, as well as another compound of concern in certain hot spots. “More sampling is needed,” Rusyn says, especially because crews working on clean-up efforts will continue excavating contaminated soil and aerating the water in local streams to extract chemicals. This could release more compounds into the air, he adds.One problem that the Texas A&M and Carnegie Mellon researchers see with the government response is a failure to communicate clearly with residents. For instance, it hasn’t been clear, Rusyn says, whether agencies on the scene have considered the effects of long-term exposure when they say a chemical level is safe.When asked what it is considering, the EPA told Nature that, in the first two weeks of an emergency response, it’s important to focus on short-term-exposure concerns — and its work can transition to longer-term concerns as time goes on.\\n\\n\\n\\n\\nA research team from Texas A&M University and Carnegie Mellon University drove a mobile laboratory around East Palestine, Ohio, on 20 and 21 February.Credit: Carnegie Mellon University/Texas A&M University\\n\\n\\nThe official reports are difficult to interpret, even for scientists, says Nicole Karn, a chemist at the Ohio State University in Columbus. “There’s already a lot of distrust among residents about whether the EPA is being truthful,” Karn says, “so [agency officials] have an uphill battle when it comes to regaining trust”.Keith Richeal, an East Palestine resident who lives five blocks from the train-derailment site, says that a strong chemical odour in the air still comes and goes, depending on the weather, and that his family has been experiencing symptoms that he thinks are related to the spill. His wife has a rash on her back, and he has been having bloody noses. He doesn’t have trust in the EPA test results because they “are all coming back the same”. He has hired an independent contractor to test the soil at his house and is waiting for results.In its statement, the EPA said that it is posting daily updates on its website and is “actively looking for ways to better illustrate the information”. The agency also has a welcome centre in town, a 24-hour hotline and weekly open houses at which residents can ask questions, it said.Getting answersThis isn’t the first time independent researchers have been called in to evaluate a chemical disaster. In 2014, after Flint, Michigan, switched its drinking-water source from Lake Huron to Flint River, residents noticed changes in their water’s taste, smell and appearance. Dissatisfied with statements by officials that the water was safe, they asked for a second opinion. In 2015, researchers revealed that there were dangerous levels of lead in the water, partially because of water-treatment decisions made by the city.\\n\\nThe Flint water crisis: how citizen scientists exposed poisonous politics\\nMona Hanna-Attisha, a public-health researcher at Michigan State University in East Lansing, was part of that effort. In situations like the one in East Palestine, she says, it’s important for authorities and health providers to use a trauma-informed response. “This is an entire community that has gone through a trauma,” she says. “Anxiety is negatively impacting their health. That’s something that folks on the ground have to be cognizant of” when interacting with residents.Andrew Whelton, an environmental and ecological engineer at Purdue University in West Lafayette, Indiana, agrees that East Palestine’s residents are experiencing fear. He travelled to the town in late February, after a community group, United for East Palestine, reached out to him. “They were reporting health impacts and concerns,” he says. “And they were not getting answers.”Along with colleagues and students, he has visited homes, interviewed residents and collected water samples from private wells, faucets and two local creeks. Whelton’s team and other researchers at Purdue analysed the samples and found several chemicals that could have been released following the accident, including acrolein. They are still analysing the data to determine the exact levels of the compounds and whether they are of concern.But Whelton says that the water from the creeks, in particular, felt “slimy” and had a “distinct sharp odour”. According to the EPA, water from the municipal system is safe to drink. The agency is encouraging people using private wells to drink bottled water until their wells have been tested.Richeal applauds the efforts by scientists. “It’s very important that independent researchers are doing this,” he says. He’s particularly worried about long-term effects: “You might think everything is okay right now. But what happens when you get all this rain and everything gets into the ground? There’s a lot of unknowns.”\\n',\n",
       "  'img_link': 'https://media.nature.com/lw767/magazine-assets/d41586-023-00820-9/d41586-023-00820-9_24645124.jpg'},\n",
       " {'title': 'How to cold e-mail for a PhD',\n",
       "  'summary': '<p>Nature, Published online: 20 March 2023; <a href=\"https://www.nature.com/articles/d41586-023-00786-8\">doi:10.1038/d41586-023-00786-8</a></p>I learnt the benefits of reaching out through e-mail while I was a student, now I receive many cold e-mails myself. Here’s how to write a good one.',\n",
       "  'field': 'careers,lab life,research management',\n",
       "  'content': '\\n\\n\\n\\n\\n\\n\\nI lead a national, graduate programme and receive multiple e-mails each week from prospective PhD students. I assess these applicants’ academic backgrounds in my discipline of climate science, as well as their interest in, and understanding of, the research my centre does. When I receive a well-written enquiry that ticks all the boxes, I experience a genuine feeling of joy. I’ve been known to forward such e-mails to a colleague with the simple addition of “this student is good!”.The programme I lead is part of an Australian Research Council Centre of Excellence — a research consortium of national universities, and national and international research partners. Our graduate students are enrolled at one of five member universities, but are part of our cross-institutional graduate programme, which provides a tailored curriculum of fundamental research and communication skills, professional development, mentoring and leadership opportunities.\\n\\nWhat I learnt from 700 e-mail applications\\nI have personal experience of success after sending a cold e-mail. During my PhD at Macquarie University in Sydney, Australia, I began searching for part-time employment to gain some industry experience and to supplement my scholarship. On the recommendation of one of my supervisors, I sent a cold e-mail to a local air-quality consultancy. In days, I was invited for an interview and offered a part-time role in a field that aligned nicely with my research. Not only did I earn the money I needed, but I also made connections that I continue to engage with 20 years later.As well as PhD enquires, cold e-mailing might also help with opportunities post PhD (cold e-mailing for postdoc roles can work under the right circumstances) or if you wish to discuss your research with or ask a question of a researcher you have not engaged with before.Here are my suggestions for how to write an effective cold e-mail that will get you noticed.The greetingDo not automatically start an e-mail with “Dear Sir…”. The number of e-mails I receive that do is quite disheartening. As a woman in science who has the lived experience of being the only woman in the room, and who advocates for gender equity, these e-mails break my heart.Rather than making an assumption about gender, address your e-mail directly using the person’s name. Although levels of formality differ by culture and country, it is always best to start with a formal greeting of “Dear Prof/Dr …”. In some countries academics only use the title Professor once they have reached full professor status, whereas in others it is common to use the term for all university academics. Look at the researcher’s listing on their university website and take guidance from there.Take note of how the researcher signs off — you can use this in subsequent e-mails. I always sign off as “Melissa”, and I am therefore inviting you to call me Melissa in future communication.The bodyHere you want to provide a succinct introduction of yourself and why you are interested in undertaking a PhD with this researcher and in this lab.\\n\\nDear grant agencies: tell me where I went wrong\\nDon’t summarize your full CV. Instead, start by introducing your current situation: “I have recently completed a masters in XX at YY university”, before detailing how your background aligns with the research of the person you are e-mailing.Talk about your recent research experience, and most importantly your research interests and how they line up with the PhD programme you’re applying to. Make sure you understand the specific expertise of the person you are contacting in the broader context of their field. For example, avoid saying “I am contacting you because you are an expert in the field of climate science”. The field of climate science has thousands of experts. Which area of climate science are you interested in? Why did you choose this particular researcher? Provide specific details that refer directly to the publications and projects of that person and their group.Be sure to mention the skills you have that would make you an ideal PhD candidate in this research area. For example, in my discipline we’re interested in recruiting people with programming experience, so that is one of the skills I look for. Do not send a generic e-mail to all academics you contact, tailor your e-mail to each recipient. This is more than simply inserting the title of one of their publications. You want to show evidence that you have read their work and state why it interests you.An example of this might look something like:Dear Professor ideal-PhD-supervisor,I am a master’s student in the Department of Excellent Science at Random University. I read with interest your paper on applying the sophisticated research method to advance excellent science and was particularly interested in your improvements to the sophisticated research method.My master’s thesis has focused on using very impressive data set to inform the sophisticated research method. This work has been undertaken under the supervision of Professor master’s supervisor. We have found that very impressive data set significantly improves the performance of sophisticated research method, and I plan to submit this research to the Journal of Excellent Science. My thesis, along with my CV and academic transcripts are attached to this e-mail.I am looking for PhD opportunities in the area of excellent science and write to ask if you are currently taking on new students? I am interested in exploring research opportunities that use even-more-sophisticated research method to discover new knowledge in excellent science. In addition to my skills in this area, I have a background in aligned discipline through my undergraduate degree, and I have also taken graduate-level courses in adjacent discipline.Your research group is an ideal fit for my background and interests, and I welcome the opportunity to discuss possible PhD opportunities with you.Regards,Potential PhD candidateThe attachmentsTo properly assess a PhD applicant, I need to see a CV and copies of academic transcripts, preferably in a PDF format. Universities differ greatly in the grading systems they use, and how they assign grades. A mark of 70% can mean near the top of the class in one country, and close to a fail in another. Therefore, it is also helpful to include a copy of your university’s grading criteria.Your CV should list your education and relevant employment experience, details of any independent research components from previous degrees, any publications you might have, and a list of contact details for referees.A cold e-mail might seem like an intimidating prospect with little chance of success, but if it is written with care and personalized it can open doors that you might not even have known were there.\\n',\n",
       "  'img_link': 'https://media.nature.com/lw767/magazine-assets/d41586-023-00786-8/d41586-023-00786-8_24622150.jpg'},\n",
       " {'title': 'Should <i>Nature</i> endorse political candidates? Yes — when the occasion demands it',\n",
       "  'summary': '<p>Nature, Published online: 20 March 2023; <a href=\"https://www.nature.com/articles/d41586-023-00789-5\">doi:10.1038/d41586-023-00789-5</a></p>Political endorsements might not always win hearts and minds, but when candidates threaten a retreat from reason, science must speak out.',\n",
       "  'field': 'society,human behaviour,politics,policy',\n",
       "  'content': '\\n\\n\\n\\n\\n\\n\\nIn October 2020, this journal endorsed Joe Biden for the next president of the United States (Nature 586, 335; 2020). It was not the first time we had endorsed a candidate for a country’s highest elected office — nor were we the only scientific publication to do so. Recognizing that politics is becoming more polarized, we had already published an editorial explaining why Nature needs to cover politics — in our journalism, commentary and primary research — when it relates to science, in line with our mission statement (Nature 586, 169–170; 2020).\\n\\nRead the paper: Political endorsement by Nature and trust in scientific expertise during COVID-19\\nThis week, Nature Human Behaviour publishes a study suggesting that Nature’s 2020 endorsement led many supporters of now former president Donald Trump to lose trust in science and in Nature as a source of evidence-based knowledge (F. J. Zhang Nature Hum. Behav. https://doi.org/10.1038/s41562-023-01537-5; 2023). The findings are based on a randomized experiment involving 4,260 US adults, carried out in mid-2021, during the COVID-19 pandemic, by Floyd Zhang at Stanford University in California.Supporters of both Trump and Biden were presented with the author’s summary of the messages conveyed in our October 2020 editorial. This highlighted Nature’s criticism of the way Trump had handled the pandemic and the journal’s expectation that Biden would do better. Participants were also given a screenshot of the title and first paragraph, and a link to the full text. They were then asked various follow-up questions.Participants who were Trump supporters did not view the summary favourably and, compared with Trump supporters who had been shown text on a different topic, had a lower opinion of Nature as an informed and impartial source on science-related issues facing society. The summary’s effect on Biden supporters was positive, but smaller. When participants were then prompted to read information from different sources about vaccine efficacy against new COVID-19 variants, Trump supporters who had been shown the summary of Nature’s editorial were less likely to trust Nature’s information on COVID-19, and also reported more mistrust in US scientists.This experiment builds on the literature on trust in research among people with different political allegiances. This includes the idea of confirmation bias, whereby people on different sides tend to favour evidence that supports the views they already have, while avoiding evidence that does not, and the backfire (or rebound) effect, whereby evidence that challenges a view can have the opposite effect to that intended.\\n\\nPolitical endorsements can affect scientific credibility\\nThe author acknowledges that it is just one experiment and that it is not clear whether the reported effects will be long-lasting. But the study does question whether research journals should endorse electoral candidates if one implication is falling trust in science. This is an important question, and there are, sadly, no easy answers. The study shows the potential costs of making an endorsement. But inaction has costs, too. Considering the record of Trump’s four years in office, this journal judged that silence was not an option.Nature’s October 2020 editorial was an appeal to readers in the United States to consider the dangers that four more years of Trump would pose — not only for science, but also for the health and well-being of US society and the wider world. Trump had laid waste to science and scientific institutions at home on issues from COVID-19 to climate change, and had gutted environmental regulations even in the face of increasing climate risk. At a time when the world needed to unite to deal with these and other global threats, he took an axe to international relationships, pulling the United States out of the 2015 Paris climate agreement and the United Nations science agency, UNESCO. He moved to defund the World Health Organization, and he walked away from a deal (the Joint Comprehensive Plan of Action) that the United States had carefully negotiated with Europe, China and Russia to prevent Iran’s government from enriching weapons-grade uranium. It is hard not to think of a worst-case scenario for public health, climate change or nuclear security had Trump remained in office today.We live in troubling times for research and for societies, and Nature’s endorsement for the November 2020 US election — and for Brazil’s similarly pivotal election last October — should be viewed in that context. Influential political voices are eschewing rigorous evidence and interfering with or undermining the functioning of independent judicial and regulatory bodies that rely on rigorous science and evidence. This has been noticeable in other countries, too, including Brazil, India, Hungary and the United Kingdom. It’s hard to know whether this is a long-term trend or global phenomenon, or something specific to certain places and circumstances. These are questions that researchers are investigating. Scientists are also testing strategies for ways to bridge the political divides, as Nature reported in a Feature earlier this month (Nature 615, 26–28; 2023).Nature doesn’t often make political endorsements, and we carefully weigh up the arguments when considering whether to do so. When individuals seeking office have a track record of causing harm, when they are transparently dismissive of facts and integrity, when they threaten scholarly autonomy, and when they are disdainful of cooperation and consensus, it becomes important to speak up. We use our voice sparingly and always offer evidence to back up what we say. And, when the occasion demands it, we will continue to do so.\\n',\n",
       "  'img_link': 'https://media.nature.com/lw767/magazine-assets/d41586-023-00789-5/d41586-023-00789-5_24644560.jpg'},\n",
       " {'title': 'How Latin America’s genomics revolution began — and why the field is under threat',\n",
       "  'summary': '<p>Nature, Published online: 20 March 2023; <a href=\"https://www.nature.com/articles/d41586-023-00794-8\">doi:10.1038/d41586-023-00794-8</a></p>The sequencing of two bacterial species created a legacy that could be at risk without further investment, warn the region’s genomic leaders.',\n",
       "  'field': 'developing world,genomics,plant sciences,education',\n",
       "  'content': '\\n\\n\\n\\n\\n\\n\\nDaniela Robles-Espinoza was a first-year undergraduate at the National Autonomous University of Mexico’s (UNAM’s) newly established Center for Genomic Sciences in Cuernavaca when she got a front-row seat to the dawn of genomics in her country.The year was 2005. Scientists in Cuernavaca, nestled in the highlands 50 kilometres southwest of Mexico City, led a government-funded project to sequence the genome of Rhizobium etli, a bacterium that lives on the roots of some bean plants, helping them to fix nitrogen. Mirroring a Brazilian effort at the University of São Paulo to sequence the bacterial plant pathogen Xylella fastidiosa five years earlier, the project was an ambitious attempt to build up the country’s biotechnology infrastructure, providing Robles-Espinoza and her fellow students with an opportunity to watch cutting-edge genomics at first hand.“I had this perception that if you want to do good science, you have to go abroad,” says Robles-Espinoza, who is now a group leader at UNAM’s International Laboratory for Human Genome Research in Querétaro, 200 km northwest of Mexico City. “It was kind of in the air that people want to leave. But I would say, not any more.”Two decades later, the sequencing of a bacterium no longer sounds like much of a milestone, but with a US$11.6-million grant from the São Paulo government, the project’s leaders say the X. fastidiosa research received more state funding than any other single piece of Brazilian science. The Mexican project received $2 million, also a major investment. To scientists across South and Central America, these projects represented more than just a loosening of governmental purse strings. By funding Mexican and Brazilian scientists to sequence agriculturally important bacteria, which were significant to their own economies, the projects have helped to spark the region’s biotech revolution.The X. fastidiosa work “created a network and a buzz within the São Paulo community. That was really important”, says Lygia da Veiga Pereira, director of the National Laboratory for Embryonic Stem Cell Research at the University of São Paulo. “The greatest impact was in making these tools more available — not just sequencing but also molecular biology.”\\n\\nCollection: Young scientists\\nBoth Mexico and Brazil’s initial, time-limited sequencing projects also helped to train a new generation of genomics researchers, says Rafael Palacios, one of the directors of the R. etli sequencing effort and coordinator of UNAM’s International Laboratory for Human Genome Research. Mexico created a specialized undergraduate programme for future geneticists. And, in Brazil, the number of students earning master’s- and doctoral-level degrees in science doubled between 2000 and 2008 (see go.nature.com/3rfim1e).Yet without greater investment, those gains are in danger. Although some large institutes have secured sustainable funding, not all researchers are so lucky. Brazil’s former president Jair Bolsonaro repeatedly cut science funding during his administration; in 2021, Brazil’s science budget was 4.4 billion reais (US$852 million), compared with 14 billion reais in 2015, the country’s peak. Researchers hope that Brazil’s newly elected president, Luiz Inácio Lula da Silva, who took office on 1 January, will reverse this trend. Mexico invests less than 1% of its gross domestic product in scientific research1 (compared with more than 3% in the United States and nearly 5% in Israel), and according to the World Bank, Mexico has just 327 scientists for every one million people (compared with 4,821 per million in the United States). Still, both Brazil and Mexico host small but vibrant biotech communities, says Palacios.“There are very good scientists doing very good genomics in Mexico,” says Palacios. “They are the ones that will change the image of Mexican genomics.”International sprintFrom his office in sprawling São Paulo, Fernando Reinach watched a revolution unfold.By the mid-1990s, US and European researchers had begun making huge strides in genome sequencing — first the laboratory bacterium Escherichia coli, then the nematode worm, the fruit fly and a flowering plant. At the same time, two international teams were racing to complete the human genome. As the biochemistry coordinator at the São Paulo Research Foundation (FAPESP), the scientific funding agency for the state of São Paulo, Reinach wanted to spark a revolution of his own. But this one would be completely Brazilian.“In these new areas where fields were booming, it was clear that Brazil was being left behind,” says Reinach. “There were just two to three people in Brazil who were able to sequence a genome.”\\n\\nBrazil election: Scientists cheer Lula victory over Bolsonaro\\nThe country had geneticists and bioinformaticians, but most of them trained abroad and in many cases stayed there. Those who returned often collaborated on European or US projects rather than leading their own. Fuelled by optimism and a desire to prove his country’s scientific mettle on the global stage, in 1997 Reinach invited 191 scientists from across São Paulo to participate in the sequencing and analysis of the bacterium X. fastidiosa.The bacterium causes citrus variegated chlorosis, a disease that has infected more than 100 million citrus trees in Brazil since 1987. Yet, although scientists had begun to tackle human-associated bacteria, no one had yet sequenced a plant pathogen. Previous work had shown that X. fastidiosa had a genome of 2.7 million base pairs. That placed it in the genomics Goldilocks zone: enough DNA to challenge scientists but small enough to be manageable.Steven Lindow, a microbiologist at the University of California, Berkeley, recalls wondering about the project’s expense. “I knew it could be worthwhile, but it was also hideously expensive,” he says. But in less than two years, the Brazilian team had sequenced, annotated and analysed the X. fastidiosa genome, on time and under budget. (It took a team of researchers five-and-a-half years to sequence E. coli2.) The resulting paper3 was published on 13 July 2000, just months after the human genome, and made the cover of Nature. João Carlos Setubal, a bioinformatics researcher at the University of São Paulo who co-led the project, says this might have been a first for Brazilian scientists.“The simple fact that a group of Brazilian scientists were able to organize a project, carry it out according to plan, and publish their results in one of the best scientific journals in the world — those are things that back then almost never happened,” Setubal says.\\n\\n\\n\\n\\nJoão Carlos Setubal is a bioinformatics researcher at the University of São Paulo, Brazil.Credit: Aldrey Olegario\\n\\n\\nLindow says the project “changed the field” by providing key insights into the complex genomics and metabolism of pathogens such as X. fastidiosa, which have to survive both their plant host and an insect vector. The fact that such a discovery came from Brazilian scientists was a bonus, he says.“Before that genome, there were very few research papers from Brazil,” adds Leonardo De La Fuente, a plant pathologist at Auburn University in Alabama, who was born in Uruguay. “That motivated a lot of people to work more in biology.”Nitrogen fixationNearly 7,500 km north of São Paulo in Cuernavaca, Palacios was conscious that both genomic sequencing and computer science in South and Central America was lagging decades behind the United States and Europe.After completing his PhD at UNAM in 1970, Palacios began a postdoctoral fellowship at Stanford University in California. Working abroad was a rite of passage for many Mexican scientists at the time, and his stint at Stanford opened his eyes to what scientists could accomplish with the right support. In 1974, Palacios returned to Mexico to dedicate his career not just to genomics, but also to improving his country’s scientific infrastructure “to help mould the people that would be the future leaders of genomic sciences in Mexico”.Six years later, he was tapped to co-lead UNAM’s Nitrogen Fixation Research Center in Cuernavaca. The institute’s immediate goal was to learn more about Rhizobium. In 2000, Palacios’ team learnt that the R. etli genome was segmented, with one-third sitting on six large circular DNA molecules called plasmids4. This partitioning, Palacios and his colleagues reasoned, provided genetic and metabolic flexibility that enabled the species to survive a range of challenging environments.\\n\\nFrustration builds over lengthy delay in revamping Mexico’s science law\\nStill, by the early 2000s, no one had attempted to sequence the organism. Palacios and his colleague Guillermo Dávila, a genomics researcher at UNAM, proposed the project to Mexico’s National Council of Science and Technology (CONACYT), Mexico’s equivalent of the US National Science Foundation, and in 2004, received a $2 million grant. Genomics laboratories across Mexico with appropriate expertise could apply to receive a state-of-the-art DNA sequencer with no restrictions on how it was to be used. The scientists would ‘repay’ the equipment cost by sequencing a certain number of bases of the R. etli genome.Published in 2006, the completed R. etli sequence revealed the complex arrangement of genes that were needed for nitrogen-fixing bacteria to convert an inert gas into usable nutrients5. But the masterpiece for Palacios and Dávila was the establishment of a four-year undergraduate degree in genomics at UNAM in 2003, which is taught at its Center for Genomic Sciences. From day one, the students would be immersed in a hands-on, research-intensive course on everything from genetics and molecular biology to statistics and computer science. A small cohort of students were admitted each year, but Palacios wanted the programme to educate Mexico’s future genomics leaders. More than three-quarters of its graduates — totalling nearly 300 so far — are now developing their careers in Mexico, after some moved abroad to gain experience after graduating.Homeward boundRobles-Espinoza was one of the first students to be enrolled in the UNAM programme. “It was very exciting as a young student to see that all of these breakthroughs can happen around me,” she says about the genome sequencing she saw in her home country. “It’s like, ‘I can do this, too!’”Some of the world’s leading geneticists gave talks to the students, she says, and after a research trip to the United Kingdom in 2009, when Robles-Espinoza began to work in the field of cancer genetics, she gained a doctorate position at the University of Cambridge, UK, and graduated in February 2015. After completing a postdoctoral fellowship, Robles-Espinoza returned to Mexico as a junior researcher at Palacios’ newest facility, UNAM’s International Laboratory for Human Genome Research.Her work now focuses on a type of melanoma that has long been overlooked by US and European scientists. Known as acral lentiginous melanoma, the cancer comprises only a few percent of melanoma cases in people of European descent but is much more common in darker-skinned populations, including those in Mexico. Because the cancer is common locally, she has ready access to patient samples. She has also teamed up with scientists at the Karolinska Institute in Stockholm.“In many cases, the priorities of a project are set by the countries who are leading it,” says Robles-Espinoza. Although Mexican scientists frequently collaborated with colleagues around the world, they weren’t the ones setting the agenda, nor were they receiving investments in training and infrastructure. “It was important to see that you can not only participate in what others are doing, you can lead and direct it.”\\n\\n\\n\\n\\nDaniela Robles-Espinoza (right) studies genetic predisposition to skin cancer.Credit: Jesús René Wong\\n\\n\\nOne of Robles-Espinoza’s classmates, Maria Gutierrez-Arcelus, also credits the UNAM programme with her career trajectory. An immunogeneticist at Boston Children’s Hospital and Harvard Medical School in Boston, Massachusetts, Gutierrez-Arcelus studies the genomic contributions to autoimmune diseases such as systemic lupus erythematosus, asthma and arthritis. Other alumni study everything from ancient DNA to plant breeding, she says.“The [sequencing] project has really promoted genomics in Mexico, and I think a lot of credit should go to Dr Palacios for having that vision,” Gutierrez-Arcelus says.Indeed, both the Brazilian and Mexican teams say that their greatest impact was in inspiring young scientists. Of the X. fastidiosa project, Setubal says, “It was very clear that the project was to be a platform on which researchers in São Paulo would get up to speed on genome sequencing technology.”And that impact extends far beyond Mexico and Brazil. As a young graduate student at the University of Chile in Santiago in the mid-2000s, Alvaro Lladser often questioned his ability to do groundbreaking science in his home country. A three-year postdoctoral fellowship at the Karolinska Institute working on cancer vaccines reinforced his doubts.“The current technology is always more advanced abroad than at home. We’re always a little bit behind,” he says.Lladser’s work depends on single-cell sequencing, a technology that wasn’t readily available when he returned to Chile in 2010 to study the tumour microenvironment at the Science and Life Foundation in Santiago. But his knowledge of what the Mexican and Brazilian teams did during the previous decade told him that he could help to build this type of capacity in Chile. He has focused on the country’s genome analytics capacity in areas such as single-cell RNA sequencing by hosting courses for fellow Chilean scientists.Facing realitiesThe new generation of scientists who trained on these early sequencing projects are helping Pereira to run a biotech start-up called gen-t, which she launched in 2021. The goal of gen-t is to provide researchers from pharmaceutical companies and academia with the as-yet-unsequenced genetic diversity in Brazil’s population. Historically, other nations have plundered Brazil’s natural resources, which has made it hard for Brazilians to benefit from their own biodiversity. Pereira hopes gen-t will help Brazilians to share the profit from their genomic diversity, and like Robles-Espinoza, help to develop biomedical solutions for the country’s heterogeneous population, which includes people of European, African and Native American descent. Less than 1% of all genomic data includes people from Central or South America6. The value of gen-t’s goal is reflected in the funding it secured in 2021, valued at 10 million reais.\\n\\nNatureTech hub\\n“This is an opportunity because Brazil has something to offer the world in our genetic diversity,” Pereira says.Today, however, funding and political realities have taken some of the shine off Setubal’s and Palacios’ original ideals. Palacios says that even if he could find the money for a one-off biotech moonshot project like he did in the early 2000s, without continued government investment, Mexico’s biotech community is at risk of floundering. Nor does Brazil have enough professorships and other permanent positions to keep its best and brightest researchers in the country, say Reinach and Setubal, and job opportunities and more lucrative funding elsewhere in the world have made it hard to keep the talent that they train. It’s one of the reasons Gutierrez-Arcelus says she has remained in Boston rather than returning home.“It’s important to be able to study all populations in the world. Having genomics well developed in Mexico can really benefit both Mexicans and the rest of the world,” she says.The sequencing of X. fastidiosa and R. etli showed the world just how much scientists in South and Central America could achieve with proper funding. The region’s political will to build on those early investments will determine how well future generations can maintain the progress that has already been made.“Young people are the future,” says Palacios.\\n',\n",
       "  'img_link': 'https://media.nature.com/lw767/magazine-assets/d41586-023-00794-8/d41586-023-00794-8_24636856.jpg'},\n",
       " {'title': 'The future of museums and a history of ignorance: Books in brief',\n",
       "  'summary': '<p>Nature, Published online: 17 March 2023; <a href=\"https://www.nature.com/articles/d41586-023-00787-7\">doi:10.1038/d41586-023-00787-7</a></p>Andrew Robinson reviews five of the best science picks.',\n",
       "  'field': 'arts,culture',\n",
       "  'content': '\\n\\n\\n\\n\\n\\n\\n\\nThe Museum of Other PeopleAdam Kuper Profile (2023)This fascinating history by anthropologist Adam Kuper discusses ethnology museums, mainly in Europe and the United States, established during the colonial period. He argues that it is time to turn such institutions into ‘cosmopolitan museums’ that include challenging perspectives and contrasting points of view — backed by research and scholarship, not “mystical insight” or the “authority of identity”. But he also recognizes that “the force is with those who demand the restitution of colonial collections”.\\n\\n\\n\\n\\n\\n\\n\\n\\nIgnorancePeter Burke Yale Univ. Press (2023)Having published a study of polymathy in 2020, cultural historian Peter Burke now tackles its opposite: ignorance. Both have long been key to scientific progress. “Thoroughly conscious ignorance is the prelude to every real advance in science,” remarked physicist James Clerk Maxwell in the nineteenth century. Chapters also consider the relevance of ignorance to business, geography, politics, religion and war. Burke argues, with encyclopedic cogency, that we should think of ‘knowledges’ and ‘ignorances’ — plural rather than singular.\\n\\n\\n\\n\\n\\n\\n\\n\\nInvention and InnovationVaclav Smil MIT Press (2023)As an environmentalist and energy writer, Vaclav Smil is well placed to analyse the impact of past and promised inventions and innovations. He distinguishes between these concepts: innovation, he says, involves “mastering new materials, products, processes and ideas”. He focuses engagingly on three types of “failed” invention: welcomed but then unwelcome (for example, leaded petrol and the pesticide DDT); over-hyped (such as nuclear fission and supersonic flight); and undelivered (including travel by vacuum tube and controlled nuclear fusion).\\n\\n\\n\\n\\n\\n\\n\\n\\nWhat Is Intergenerational Justice?Axel Gosseries Polity (2023)What duties do we have to future generations, both living and unborn? What does ‘justice’ mean in the intergenerational context? On topics such as climate change, we have to “divide cakes without knowing how many guests will join us and what their tastes will be”, comments political philosopher Axel Gosseries. Despite the emotive subject, his language and tone are academic. Greenhouse gases and extinction are discussed, but not Greenpeace and Extinction Rebellion — even though he accepts the need for “radical changes”.\\n\\n\\n\\n\\n\\n\\n\\n\\nThe OneHeinrich Päs Basic (2023)Theoretical physicist Heinrich Päs begins his intriguing, controversial book for general readers on the mysteries of quantum physics by asking, how can standing under a star-studded night sky make us feel at once insignificant yet “strangely at home in the universe”? He answers, essentially, that everything — including quantum particles — is part of the same fundamental whole. He is a monist, convinced by a comment by ancient Greek philosopher Heraclitus: “From all things One and from One all things.”\\n',\n",
       "  'img_link': 'https://media.nature.com/lw767/magazine-assets/d41586-023-00787-7/d41586-023-00787-7_24626162.jpg'},\n",
       " {'title': 'The brain science collaboration that offers hope to blind people',\n",
       "  'summary': '<p>Nature, Published online: 17 March 2023; <a href=\"https://www.nature.com/articles/d41586-023-00819-2\">doi:10.1038/d41586-023-00819-2</a></p>A key aim of Pieter Roelfsema’s research is to develop a device to restore rudimentary eyesight to people whose optic nerve has died.',\n",
       "  'field': 'brain,careers,neuroscience',\n",
       "  'content': \"\\n\\n\\n\\n\\nNeuroscientist Pieter Roelfsema describes how his research might ultimately help people who have lost their sight.\\n\\nYour browser does not support the audio element.\\n    \\n\\n\\nDownload MP3\\n\\nSee transcript\\n\\n\\n\\nAn applied goal of Pieter Roelfsema’s lab at the Netherlands Institute for Neuroscience in Amsterdam is to create a visual brain prosthesis aimed at people who have lost their sight.To help achieve this goal, the lab partners with both neurosurgeons and artificial intelligence researchers.“We are knowledgeable about how to put electrodes in the brain,” says Roelfsema, “but we collaborate with experts who know about how to make these electrodes so that they don't damage the brain tissue too much, also with people in artificial intelligence who can take camera images and translate them into brain stimulation patterns.“We also collaborate with neurosurgeons who can inform us how to really make this device and make it something that is going to be feasible for a neurosurgeon to really implant in the brain. That is definitely a very important goal for me, to bring this to a patient.”In episode five of Tales from the Synapse, a podcast series with a focus on brain science, Roelfsema describes how he handles requests from people who are pinning their hopes on being able to see again. “I have to explain this is not a clinically approved device,” he says.“Our ambition will be to go to humans in the next say, two years, or maybe a little bit later, but it’s still going to be research. There are all kinds of regulations, which are there for a good reason. And we have to show that we comply with all these regulations.”Tales from the Synapse is produced in partnership with Nature Neuroscience and introduced by Jean Mary Zarate, a senior editor at the journal. The series features brain scientists from all over the world who talk about their career journeys, collaborations and the societal impact of their research.\\n\",\n",
       "  'img_link': 'https://media.nature.com/w767/magazine-assets/d41586-023-00819-2/d41586-023-00819-2_24026100.jpg'},\n",
       " {'title': 'Five things I wish academia understood about my social anxiety',\n",
       "  'summary': '<p>Nature, Published online: 17 March 2023; <a href=\"https://www.nature.com/articles/d41586-023-00830-7\">doi:10.1038/d41586-023-00830-7</a></p>I’ve learnt to overcome some aspects of my social anxiety, but I hope sharing my experiences can help others recognize the difficulties faced by scientists like me.',\n",
       "  'field': 'careers,lab life,communication',\n",
       "  'content': '\\n\\n\\n\\n\\n\\n\\nOver the years, I’ve met numerous people who have admitted to experiencing social anxiety in some form. Often, their struggles are especially acute in academic environments, in which ‘flawless’ interactions with supervisors, colleagues, peers and interview panels are crucial to success.I can relate to the experiences of these individuals on a personal level, because as a child I had selective mutism. Children with this social-anxiety disorder are unable to speak in some situations or around certain people, despite being perfectly capable of speech in other contexts. Left untreated, it often persists into adulthood.While at school, I wouldn’t drink any water during the day to avoid having to ask to go to the bathroom. As an undergraduate, I got used to getting a zero when marks were awarded for participation, and I fretted over oral presentations for weeks beforehand.Being involved in field research at university was what eventually helped me to overcome my mutism. I was so excited about the plants and insects I was working with that I could no longer keep quiet. Instead, I found myself talking freely with my teachers and fellow students about the flora and fauna we were seeing. Now, as an ecology graduate student, I love many aspects of research. I feel fortunate to have landed myself a position in a laboratory with a supportive supervisor, and to have had opportunities to conduct fieldwork in stunning landscapes.Tricky situationsHowever, fieldwork is only one part of my life in research. There are so many situations in which social anxiety can surface — from conferences and interviews to lab meetings and social events. Although I no longer experience mutism, intense feelings of anxiety persist in these sorts of circumstance, and have shaped my life as a young academic. At times, my anxiety has left me feeling inadequate, misunderstood, isolated and uncertain about pursuing a career in academia. On the basis of conversations with other students, I don’t think my experiences are uncommon.\\n\\nHow burnout and imposter syndrome blight scientific careers\\nI find that people often misread my behaviour or demeanour because they are unaware of the ways in which social anxiety can manifest itself. Here are five examples:1. I struggle with conversations that involve being face-to-face, especially in public venues, so having an informal discussion in a café or restaurant is difficult for me. But if you were to join me for a hike, or chat to me in a quiet meeting room, I’d find it much easier. I’ve also had fruitful scientific conversations while working in the lab and in the field, and even while washing dishes after an event.2. Looking into people’s eyes can be unsettling for me, and when I try to do it, I have a hard time concentrating on what is actually being said. Please don’t interpret my lack of eye contact as disinterest or boredom. I might not be looking at you, but I am listening, engaged and empathetic.3. If I’m silent, it doesn’t mean I’m being rude, unfriendly or judgemental. I know it can be difficult to get a sense of what I’m thinking if I’m quiet, but in some situations it’s hard for me to speak my thoughts. Ideas become jumbled in my mind, and by the time I’ve settled on how to communicate them, the conversation has moved on.4. Anxiety isn’t ‘rational’, so I can’t reason my way out of it, or identify logical patterns that characterize it. My anxiety manifests itself intensely in some situations, but not at all in others, in seemingly inconsistent ways. For me, oral presentations are no longer a problem, but attending academic social events can be overwhelming.5. I want to connect with people but, ironically, my anxiety often emerges when I’m in the company of people with whom I particularly want to have intellectual and meaningful discussions. I can happily strike up conversations with strangers and chat amicably with the public at outreach events, but become silent, awkward or avoidant when I’m with the people I most want to talk to. This is an isolating experience, and I hope that those who experience my difficulties in communicating will not take it personally.Work in progressAs a doctoral student, I chip away at my thesis every day; I make research plans, seek out help if I need it, set goals and do my best to achieve them. I try to deal with my social anxiety in a similar way, and I recognize my own responsibilities in this journey. I also do my best to make sure my anxiety doesn’t interfere with or inconvenience those around me. I understand that, for many, cafés are wonderful places for scientific exchanges, and the last thing I want is for my condition to deter others from enjoying useful discussions over coffee.\\n\\nCollection: Science careers and mental health\\nAt the same time, I wonder whether we can work towards a middle ground as an academic community. In his 2017 book Academic Ableism: Disability and Higher Education, Jay Dolmage, a disability-rights activist based at the University of Waterloo in Canada, describes the notecard technique, whereby people attending seminars, lectures and conferences can submit questions or comments to speakers on paper or digitally. I love this idea. Putting a hand up in this setting can be daunting — my notebooks are regularly filled with unasked questions and unshared thoughts. Notecards give audience members a chance to ask questions in a non-verbal manner. The pandemic has led to the development of a wealth of tools that facilitate this, including Slido, Mentimeter, Vevox and even Zoom’s chat function.Various other ideas, such as anonymous polling and small-group ice-breakers, have been proposed as ways to make university classrooms less anxiety-inducing1. As a teaching assistant, I’ve taken to encouraging reticent students in my tutorial and lab sessions to e-mail me if they have any questions or concerns.I know that the experiences of people struggling with social anxiety in academia vary widely. Because of this, I doubt there is a single practical solution with which to address it. That said, my personal wish, above all, is for greater understanding and acceptance in the academic community. Those of us who falter in conversation, avoid social events and look at the ceiling when we speak tend not to give good first impressions. We might seem unfriendly and disengaged — perhaps even judgemental and haughty. I sincerely hope that what I’ve shared testifies otherwise.\\n',\n",
       "  'img_link': 'https://media.nature.com/lw767/magazine-assets/d41586-023-00830-7/d41586-023-00830-7_24597586.jpg'},\n",
       " {'title': 'Strife at <i>eLife</i>: inside a journal’s quest to upend science publishing',\n",
       "  'summary': '<p>Nature, Published online: 17 March 2023; <a href=\"https://www.nature.com/articles/d41586-023-00831-6\">doi:10.1038/d41586-023-00831-6</a></p>Editors threaten to resign over ‘no-reject’ model that others see as the future of research journals.',\n",
       "  'field': 'publishing',\n",
       "  'content': '\\nLast October, the pioneering life-sciences journal eLife introduced bold changes to its editorial practice — which some researchers applauded as reimagining the purpose of a scientific journal. From 31 January this year, eLife said, it would publish every paper it sent out for peer review: authors would never again receive a rejection after a negative review. Instead, reviewers’ reports would be published alongside the paper, together with a short editorial assessment of the work’s significance and rigour. Authors could then decide whether to revise their paper to address any comments.The change followed an earlier decision by eLife to require that all submissions be posted as preprints online. The cumulative effect was to turn eLife into a producer of public reviews and assessments about online research. It was “relinquishing the traditional journal role of gatekeeper”, editor-in-chief Michael Eisen explained in a press release, and “promoting the evaluation of scientists based on what, rather than where, they publish”.\\n\\n\\n\\n\\n\\nThe transformation sparked enthusiastic praise — and sharp criticism. Some scientists saw it as a long-overdue move to empower authors. Others, including some of eLife’s academic editors (who are mostly senior researchers), weren’t so happy. They worried it would diminish the prestige of a brand they’d worked hard to build, and some wrote privately to Eisen (in letters seen by Nature) to say they would resign if the plan was fully implemented. Amid the pushback, the journal postponed switching fully to its new process.But the dispute only heightened. On 9 March, 29 eLife editors — including the journal’s former editor-in-chief, Randy Schekman — wrote to Damian Pattinson, executive director of the journal’s non-profit publisher, eLife Sciences Publications in Cambridge, UK, asking that Eisen be replaced “immediately”. They added that they had no confidence in Eisen’s leadership, because he had dismissed their concerns and had not considered compromise positions. One of the journal’s five deputy editors had already stepped down from that leadership position, and “significant numbers” of reviewers and senior editors were “standing ready to resign”, they wrote.Eisen, a Howard Hughes Medical Institute (HHMI) investigator who works at the University of California, Berkeley, fired back publicly online, tweeting on 12 March that academics were “lobbying hard to get me fired”. He later deleted the tweet, but told Nature in an interview that “opposition to eLife’s model is driven fundamentally by powerful scientists not wanting to change a system that has benefited them and which they have sculpted to continue to reward them”. In response, Schekman and other authors stated that Eisen’s comments were “not true and do not reflect our legitimate concerns with the new model at eLife”.Eisen says he thinks the dissent is small in scale. He and Pattinson say they did not dismiss concerns, but consulted on changes over two years with editors. “We see big swathes of enthusiasm among the community,” Pattinson adds.The row highlights disagreements among researchers about the function of journals and peer review — and, potentially, about the future of science publishing. Some eLife editors argue that journals should use review to guide filtering and rejection of papers. But supporters of eLife’s changes see benefit in preventing peer review from serving as a prestige-gathering function, in which, by rejecting most of the manuscripts submitted to them, selective journals become perceived as arbiters of what work matters. “We rely too much on journal titles in judging people’s work,” Eisen says. “If we want to fix a bad system, we do have to break some eggs.”What is a journal’s purpose?When eLife was launched in 2012 with the financial backing of three powerful science funders — the Maryland-based HHMI, the UK Wellcome Trust and Germany’s Max Planck Society — it had the aim of being a non-commercial and academic-edited journal that would rival prestigious titles such as Cell, Nature and Science. (Nature’s news team is editorially independent of the journal’s research team.) Besides being open access, another of its key innovations was a collaborative system of peer review, where referees and a handling editor discuss comments together. The journal attracted dozens of working scientists as editors who triage submissions, with hundreds more scientists as reviewing editors.\\n\\nOpen-access journal eLife announces ‘preprint first’ publishing model\\neLife had its eye on bigger changes, however. In 2021, the journal decided to publish only papers that were already preprints. This meant that delays in reviewing wouldn’t hold up an author from sharing their work. But even before Eisen and Pattinson joined, the journal had run a trial with more than 300 manuscripts to test the idea of ditching rejection after review. Its aim was to simply publish papers with reviews, author responses and editorial ratings. “The peer-review process does not need to end with a binary outcome of acceptance or rejection,” the journal wrote in a 2019 analysis of that work.It was this idea that eLife instituted for all papers last October, with the addition that editors would also append a short summary assessment of the paper — giving readers a quick idea of its quality and significance. “This puts power back in the hands of the authors, who can then publish what they have, instead of having to do ever more experiments to satisfy reviewers,” says Eisen. The journal plans to charge US$2,000 for the process of arranging review on submissions; previously, its open-access publication fee was $3,000.Some eLife editors are fully on board with the new system. “It’s the future, where science is going,” says senior eLife editor Panayiota Poirazi, a neuroscientist at the Institute of Molecular Biology and Biotechnology in Heraklion, Crete. Among the journal’s funders, HHMI says it fully supports the new policy. Wellcome says that it supports eLife’s publishing process, and the Max Planck Society told Nature it was still discussing the issue.But other researchers have been openly critical from the start. In November, 47 editors wrote privately to Eisen asking for a rethink or for more time to experiment — perhaps running the new system alongside the conventional one, or creating a second journal in which to publish papers of less significance. They worried about harm to the journal’s collaborative open-reviewing process, and that the quality of papers on the eLife platform would drop. With no possibility of rejection, some authors might choose to ignore reviewer comments or only superficially address them, they wrote — and that knowledge might discourage reviewers from producing detailed critiques. Responding to these concerns, Eisen and Pattinson say that they haven’t seen such problems so far, although the project is in its early days, and that running two systems would reduce the chances of the new model’s success.Editors also argued that removing rejection-after-review meant more pressure on the gatekeeping step that remains in eLife’s system — the triage point where editors choose whether to send out a paper for review. That step had been “opaque and subject to errors in judgment”, their letter stated, an issue that would become more consequential if later negative reviews could no longer lead to rejection. Editors might react by becoming more conservative and decide not to take a chance on manuscripts from less-well-known authors. But Eisen says that, in the new system, sending a preprint for review shouldn’t communicate anything about its quality or importance: the reviews and editorial assessments do that instead. The guidance that editors should follow when deciding what to send for review is “can you generate high-quality and broadly useful public reviews of this paper?”, he says.\\n\\nWill the pandemic permanently alter scientific publishing?\\nIn some countries, hiring and promotion decisions still rely heavily on journal titles in candidates’ publication lists — something that is unlikely to change quickly, the editors added in their letter. They worried that scientists there would stop sending their manuscripts to eLife. Eisen, however, says that problematic reliance on journal titles will continue until there is an alternative system, such as eLife’s.In a further private letter sent to Eisen in January, 30 editors said they would resign once the new policy was fully implemented.The full scale of the discontent is unclear. Although Eisen and Pattinson say they’ve had broad support, Axel Brunger, a structural biologist at Stanford University in California, who initiated the first letter, says he reached out only to his colleagues in structural biology and neuroscience, and that nearly all agreed to sign up. “The concerns are widespread,” he says.One researcher who signed all three letters is neuroscientist Gary Westbrook at the Vollum Institute at Oregon Health & Science University in Portland. He is a vocal critic of what he sees as the monopoly that commercial journals have in science publishing, and says he signed “because I didn’t think the new policy was realistic”. Far from helping eLife as a non-profit, high-quality alternative, he says, he thinks the model will diminish its impact.Reviewing preprintsThe concept of reviewing preprints is catching on in the life sciences. At least two dozen preprint-refereeing initiatives of various sizes have been launched in the past few years. The largest (apart from eLife itself) is Review Commons, launched in December 2019 by the California-based non-profit organization ASAPbio and EMBO Press. The latter runs five journals and is part of the European Molecular Biology Organization in Heidelberg, Germany. As a review-sharing collaboration between 17 journals from 6 publishers, including eLife, Review Commons uses EMBO Press editors to select referees for submissions. Authors can ask Review Commons to post reviews and any further author responses on a preprint server, or they can submit their paper, with reviews and responses, to any journal. More than 2,000 reviews of 540 articles have been run through this system.The idea of ‘journal agnostic’ reviewing is still at proof-of-principle stage, says Bernd Pulverer, EMBO’s head of scientific publications. But he sees merit in having both peer-reviewed preprints and conventional journals, which, he says, provide “real added value in condensing and stratifying information”.That view is shared by Maria Leptin, president of the European Research Council. “If I want to learn about a new field that is not core to my own, then I want a trustworthy source that filters for general interest,” she says. “eLife now does its filtering upstream, in a non-transparent, unaccountable way.”The triage stage shouldn’t be seen as this kind of filter, says Eisen. “People are used to operating in a world where appearance in a journal tells you about the quality, audience or import of a study. This is precisely what we are trying to change,” he says. He argues that the short editorial summary eLife appends to its articles serve as quality guides for readers. They grade the significance of the findings (landmark, fundamental, important, valuable, useful) and assess the strength of their support (exceptional, compelling, convincing, solid, incomplete, inadequate).More consultation?Endocrinologist Mone Zaidi at Icahn School of Medicine at Mount Sinai, New York City, is one of eLife’s four remaining deputy editors and has been trying to mediate the issue. He admires Eisen’s vision, he says, “but any new, transformative change has to be done in a careful manner, with buy-in from the community”.Together with some of his colleagues, he is trying to persuade Eisen to slow down, to avoid mass resignations and to establish milestones to assess the effects the changes would have on the lives of working scientists. “There has to be consultation and risk-mitigation plans,” he says.The deputy editor who stood down, cell biologist Anna Akhmanova at the University of Utrecht in the Netherlands, shares Zaidi’s view. She says she helped to develop the new system, but stepped down as deputy editor because it was being pushed through too fast. “We need evolution, not revolution — many small, careful steps to try to move the community towards what would be a better publishing system,” she says.Eisen says he has already responded to concerns by extending — for a short time — the deadline for the regular reviewing system. “We expect things to evolve in interesting ways as people start to see the advantages and opportunities of not making publishing decisions.”“eLife is doing a big and interesting experiment, however it works out,” says stem-cell biologist Fiona Watt, a former eLife deputy editor who is now EMBO’s director. “My sense as a scientist is that the publishing landscape is changing again.”\\n',\n",
       "  'img_link': 'https://media.nature.com/lw767/magazine-assets/d41586-023-00831-6/d41586-023-00831-6_24636118.jpg'},\n",
       " {'title': 'How the US will remove ‘forever chemicals’ from its drinking water',\n",
       "  'summary': '<p>Nature, Published online: 17 March 2023; <a href=\"https://www.nature.com/articles/d41586-023-00822-7\">doi:10.1038/d41586-023-00822-7</a></p>The EPA has proposed a strict PFAS limit, but it will take money and innovative technologies to implement the plan.',\n",
       "  'field': 'water resources,policy,chemistry,government',\n",
       "  'content': '\\n\\n\\n\\n\\n\\n\\nThe US Environmental Protection Agency (EPA) has proposed the first limitations on a set of pervasive and dangerous ‘forever chemicals’ in US drinking water. The chemicals, known for their strong carbon–fluorine bonds, are difficult to destroy and have become widely dispersed in the environment. Scientists and engineers are busy developing ways to extract the chemicals more efficiently from water and soil and break them down, but water utility companies warn that meeting the EPA’s new standards will be expensive in the short term — possibly prohibitively so for small water-treatment facilities.“This is a huge deal, in terms of protecting public health, but also in terms of what it’s going to take to accomplish,” says Michelle Crimi, an environmental engineer at Clarkson University in Potsdam, New York.\\n\\nTainted water: the scientists tracing thousands of fluorinated chemicals in our environment\\nProposed on 14 March, the regulation targets perfluorinated and polyfluorinated alkyl substances (PFAS), a class of thousands of nearly indestructible compounds used in everything from non-stick cookware and waterproof clothes to industrial materials and cosmetics. Once called miracle chemicals for their hallmark durability, PFAS accumulate in the environment and in people; even minute amounts increase the risk of cancer, as well as the risk of developmental and other health problems1, research shows.The EPA suggested a voluntary limit for PFAS in drinking water in 2016, but this is the first time it has advanced a mandatory requirement. The core of the proposal would restrict two of the most dangerous PFAS compounds, PFOA and PFOS, to four parts per trillion. That is the lowest level that is detectable using current laboratory tests, although the agency has determined that there are risks associated with much lower concentrations. Another four chemicals would be regulated as a mixture.Similar movements to rein in PFAS are afoot internationally. At the extreme end of the spectrum, the European Union is considering legislation that would ban the production of PFAS altogether.Health costsAchieving the EPA’s proposed regulation won’t be cheap. PFAS contamination has been found in around 2,800 communities in the United States, according to the Environmental Working Group, an advocacy organization based in Washington DC, and research by the group suggests that it probably affects the water supplies of at least 200 million people2. And although the use of PFOA and PFOS has mostly been phased out in the United States, the group has identified around 30,000 industrial facilities that could be using countless other compounds in the PFAS family.\\n\\nHow to destroy ‘forever chemicals’: cheap method breaks down PFAS\\nNumerous states have already set limits on PFAS in drinking water, and water providers have demonstrated that existing technologies such as carbon filtration can reduce PFAS amounts to undetectable levels. But installing such technologies nationally could be costly, with the financial burden falling disproportionately on smaller water-treatment systems. For facilities large and small, adding PFAS filtration will have to be weighed against other priorities, such as replacing lead pipes, says Chris Moody, a regulatory analyst with the American Water Works Association (AWWA), which is based in Denver, Colorado, and represents more than 4,300 utility companies that provide some 80% of the US drinking-water supply.By one measure, the EPA estimates that implementing its proposal nationally would cost around US$772 million annually, but a study commissioned by the AWWA using similar assumptions suggests that the price tag could be around $2.9 billion a year. The EPA says that more than $9 billion is already available through a US infrastructure law enacted in late 2021, but Moody stresses that this is just a start: the AWWA-estimated cost over 20 years is $58 billion.If history is any indicator, however, costs will probably come down over time, says Melanie Benesh, vice-president of government affairs at the Environmental Working Group. “With regulation often comes market innovation,” she adds.Innovative solutionsScientists and engineers started investigating technologies years ago, when the risks posed by PFAS became clear. Research has focused on methods to more efficiently remove PFAS from drinking water, clean up groundwater contamination or destroy the chemical compounds.The upshot is that a variety of promising technologies are now available, from carbon filtration and ion-exchange systems that can separate PFAS from drinking water to electrochemical and gasification methods to break down PFAS, says Patrick McNamara, an engineer at Marquette University in Milwaukee, Wisconsin. But scaling them up to be practical could be challenging, he adds.For her part, Crimi is working with the US Department of Defense to test a technology that could be used to clean up plumes of PFAS contamination in groundwater before they leach into drinking-water supplies. Starting as early as this year at Peterson Space Force Base in Colorado Springs, groundwater will be collected inside a horizontal well and funnelled through a reactor developed by Crimi’s team that uses ultrasound waves to break the carbon–fluorine bonds in PFAS3.“We know it’s effective in the lab,” she says, but there are always things to learn when scaling up to field operations.The EPA is accepting comments on the proposal until mid-April.\\n',\n",
       "  'img_link': 'https://media.nature.com/lw767/magazine-assets/d41586-023-00822-7/d41586-023-00822-7_24636802.jpg'},\n",
       " {'title': 'GPT-4 is here: what scientists think',\n",
       "  'summary': '<p>Nature, Published online: 16 March 2023; <a href=\"https://www.nature.com/articles/d41586-023-00816-5\">doi:10.1038/d41586-023-00816-5</a></p>Researchers are excited about the AI — but many are frustrated that its underlying engineering is cloaked in secrecy.',\n",
       "  'field': 'computer science,machine learning',\n",
       "  'content': \"\\n\\n\\n\\n\\n\\n\\nArtificial intelligence company OpenAI this week unveiled GPT-4, the latest incarnation of the large language model that powers its popular chat bot ChatGPT. The company says GPT-4 contains big improvements — it has already stunned people with its ability to create human-like text and generate images and computer code from almost any a prompt. Researchers say these abilities have the potential to transform science — but some are frustrated that they cannot yet access the technology, its underlying code or information on how it was trained. That raises concern about the technology’s safety and makes it less useful for research, say scientists.One upgrade to GPT-4, released on 14 March, is that it can now handle images as well as text. And as a demonstration of its language prowess, Open AI, which is based in San Francisco, California, says that it passed the US bar legal exam with results in the ninetieth centile, compared with the tenth centile for the previous version of ChatGPT. But the tech is not yet widely accessible — only to paid subscribers to ChatGPT so far have access.\\n\\nChatGPT listed as author on research papers: many scientists disapprove\\n“There’s a waiting list at the moment so you cannot use it right now,” Says Evi-Anne van Dis, a psychologist at the University of Amsterdam. But she has seen demos of GPT-4. “We watched some videos in which they demonstrated capacities and it’s mind blowing,” she says. One instance, she recounts, was a hand-drawn doodle of a website, which GPT-4 used to produce the computer code needed to build that website, as a demonstration of the ability to handle images as inputs.But there is frustration in the science community over OpenAI’s secrecy around how and what data the model was trained, and how it actually works. “All of these closed-source models, they are essentially dead-ends in science,” says Sasha Luccioni, a research scientist specializing in climate at HuggingFace, an open-source-AI community. “They [OpenAI] can keep building upon their research, but for the community at large, it’s a dead end.”‘Red team’ testingAndrew White, a chemical engineer at University of Rochester, has had privileged access to GPT-4 as a ‘red-teamer’: a person paid by OpenAI to test the platform to try and make it do something bad. He has had access to GPT-4 for the past six months, he says. “Early on in the process, it didn’t seem that different,” compared with previous iterations.\\n\\nAbstracts written by ChatGPT fool scientists\\nHe put to the bot queries about what chemical reactions steps were needed to make a compound, predict the reaction yield, and choose a catalyst. “At first, I was actually not that impressed,” White says. “It was really surprising because it would look so realistic, but it would hallucinate an atom here. It would skip a step there,” he adds. But when as part of his red-team work he gave GPT-4 access to scientific papers, things changed dramatically. “It made us realize that these models maybe aren’t so great just alone. But when you start connecting them to the Internet to tools like a retrosynthesis planner, or a calculator, all of a sudden, new kinds of abilities emerge.”And with those abilities come concerns. For instance, could GPT-4 allow dangerous chemicals to be made? With input from people such as White, OpenAI engineers fed back into their model to discourage GPT-4 from creating dangerous, illegal or damaging content, White says.Fake factsOutputting false information is another problem. Luccioni says that models like GPT-4, which exist to predict the next word in a sentence, can’t be cured of coming up with fake facts — known as hallucinating. “You can’t rely on these kinds of models because there’s so much hallucination,” she says. And this remains a concern in the latest version, she says, although OpenAI says that it has improved safety in GPT-4.Without access to the data used for training, OpenAI’s assurances about safety fall short for Luccioni. “You don’t know what the data is. So you can’t improve it. I mean, it’s just completely impossible to do science with a model like this,” she says.\\n\\nHow Nature readers are using ChatGPT\\nThe mystery about how GPT-4 was trained is also a concern for van Dis’s colleague at Amsterdam, psychologist Claudi Bockting. “It’s very hard as a human being to be accountable for something that you cannot oversee,” she says. “One of the concerns is they could be far more biased than for instance, the bias that human beings have by themselves.” Without being able to access the code behind GPT-4 it is impossible to see where the bias might have originated, or to remedy it, Luccioni explains.Ethics discussionsBockting and van Dis are also concerned that increasingly these AI systems are owned by big tech companies. They want to make sure the technology is properly tested and verified by scientists. “This is also an opportunity because collaboration with big tech can of course, speed up processes,” she adds.Van Dis, Bockting and colleagues argued earlier this year for an urgent need to develop a set of ‘living’ guidelines to govern how AI and tools such as GPT-4 are used and developed. They are concerned that any legislation around AI technologies will struggle to keep up with the pace of development. Bockting and van Dis have convened an invitational summit at the University of Amsterdam on 11 April to discuss these concerns, with representatives from organizations including UNESCO’s science-ethics committee, Organisation for Economic Co-operation and Development and the World Economic Forum.Despite the concern, GPT-4 and its future iterations will shake up science, says White. “I think it's actually going to be a huge infrastructure change in science, almost like the internet was a big change,” he says. It won’t replace scientists, he adds, but could help with some tasks. “I think we're going to start realizing we can connect papers, data programmes, libraries that we use and computational work or even robotic experiments.”\\n\",\n",
       "  'img_link': 'https://media.nature.com/lw767/magazine-assets/d41586-023-00816-5/d41586-023-00816-5_24630796.jpg'},\n",
       " {'title': 'Sweeping report calls for anti-bias measures in US science',\n",
       "  'summary': '<p>Nature, Published online: 16 March 2023; <a href=\"https://www.nature.com/articles/d41586-023-00823-6\">doi:10.1038/d41586-023-00823-6</a></p>Universities and science employers must adopt practices to diversify their research workforce and move beyond simply hiring members of under-represented groups, finds the US National Academies of Sciences, Engineering, and Medicine.',\n",
       "  'field': 'careers,policy,scientific community',\n",
       "  'content': '\\n\\n\\n\\n\\n\\n\\nUniversities and other employers in the US science sector must adopt practices that foster a safe and inclusive community, finds a report1 from the US National Academies of Sciences, Engineering, and Medicine (NASEM), which outlines how such institutions can do so. The report calls for systemic change across multiple levels to account for the lengthy history of discrimination against people of colour and members of marginalized communities in the United States.Susan Fiske, co-chair of the committee behind the report, says that bias and discrimination in science, technology, engineering, mathematics and medicine (STEMM) is structural1 in the United States. This culture, she says, results in policies that reinforce each other and attitudes that perpetuate lifelong disadvantages for certain groups. “The converging data from surveys and experiments and observations tells us that you can’t fix it by fixing individuals,” says Fiske, who researches psychology and social policy at Princeton University in New Jersey.If universities and science organizations fail to adopt deliberate practices to diversify their research workforce, warns Fiske, they could miss out on the benefits that various perpectives can bring. Science is increasingly being pushed forward by the work of teams rather than individuals2. Diverse teams can outperform groups made up of similar people, even if those people are considered ‘top performers’3. Similarly, students who are members of under-represented groups innovate at higher rates than do those from majority groups, although their contributions are more likely to go unacknowledged — the ‘diversity–innovation paradox’4.Full potentialSystemic changes, the report says, are therefore required that go beyond simply hiring a quota of scientists who are members of minority ethnic groups: the changes must include fostering an environment in which those scientists can realize their full potential. In July 2020, Eddie Bernice Johnson, then a Democratic member of the US House of Representatives and chair of the House Committee on Science, Space, and Technology, sent a letter to NASEM in the wake of racial-justice protests that had swept the nation following the murder of Black men and women by police. The letter asked NASEM to synthesize existing research on the career challenges faced by scientists from minority ethnic groups and to brainstorm solutions. Although the report focused on the experiences of Black academics, the committee also commissioned papers on the issues facing Indigenous, Latino/Latina and Asian people.The committee highlighted the specific role of ‘gatekeepers’ — those who can permit or hinder access to resources — and what they can do to combat discrimination. But, the report notes, such individuals might themselves have subconscious biases. “You can’t be counted on to notice when you’re being biased,” Fiske says, adding that institutions should build expectations related to diversity, equity and inclusion (DEI) into job descriptions. Institutions, she says, should also adopt oversight policies that track gatekeepers’ decisions, including the recruitment, advancement and retention of students and staff members, and the reasons that people choose to leave. An individual’s bad experience might not amount to a discriminatory environment, she says, but if several people report that they had faced challenges, it could signal that an institution has failed to create a space in which scientists from under-represented groups feel safe and supported.In addition, those who set priorities, such as principal investigators (PIs), have the capacity to create equitable spaces for scientists by setting examples of good behaviours and promoting an environment of psychological safety. Hiring more scientists who are members of under-represented groups avoids tokenizing any one person, and the report suggests that PIs set unifying goals for their team in which everyone’s role is clearly defined, celebrate the achievements of individual scientists, and downplay expectations of dress and appearance that cater to a majority population. Striving for an egalitarian atmosphere in which each team member has the ability to share input or feedback on their work can also lessen power differentials, especially if the team leader is from the ethnic or racial majority, the report notes. The authors also cite research showing that diversifying all levels of management increases group productivity5.For Jesse Lee, a fifth-year PhD student in Gregory Beatty’s cancer-immunology laboratory at the University of Pennsylvania in Philadelphia, this type of group structure has led to what he calls “one of the more healthy lab environments” that he’s been a part of. Lab technicians are each trained in specialized techniques, such as mouse surgery or histology, and several technicians will collaborate on a project, training students as they go. This, coupled with Beatty’s hands-on approach to mentoring, means that “you’re constantly working together and coordinating how your things run through each team”, Lee says. “Everybody knows a little bit about everybody’s projects, and everybody’s always thinking about someone else’s project.”Inclusive communitiesThe committee acknowledges ongoing work to foster inclusive communities by institutions — including historically Black colleges and universities (HBCUs) — that serve members of under-represented groups, and suggests that other institutions look to them as guides and partners. M. Roy Wilson, the president of Wayne State University in Detroit, Michigan, and a member of the committee, says that the culture of HBCUs goes beyond a student or faculty member simply seeing more people on campus who look like them. “If you talk to students who go to HBCUs, the things that they say about their institutions aren’t things I typically hear from other types of institutions,” he says, adding that any such comments probably relate to the resources, opportunities and mentorship that students receive.Carl Pettis, the provost for academic affairs at Alabama State University (ASU), an HBCU in Montgomery, says that the university begins reaching out to potential students at a young age — from kindergarten, or age 5, through secondary school. ASU has also developed a pipeline to recruit students from community colleges that offer two-year associate’s degrees. At ASU, students receive mentoring not just from faculty members, but from peers as well. The report recommends that universities team up with HBCUs to learn from their processes, and Pettis says that ASU is always on the lookout for new partnerships. “Gone are the days where larger institutions are simply reaching out to show a little diversity in their proposals,” he says. “Now, you’re signing on with an HBCU because they are a quality partner and they provide quality students.”\\n\\nCollection: Research leadership\\nThe report represents several firsts for NASEM, including the integration of lived experiences of Black scientists alongside experimental data. “We’ve been forced to look at racism, including structural racism, and the effect that it has had on educational disparities,” says Marcus Lambert, a workforce-diversity researcher and associate vice-president for research strategy and operations at SUNY Downstate Health Sciences University in New York City . “Ultimately,” he says, “I think this will be a go-to resource for our STEMM community for many years to come”.To complement data gleaned from an exhaustive literature search, the committee worked closely with NASEM’s Roundtable on Black Men and Black Women in Science, Engineering, and Medicine to conduct structured interviews about their lived experiences. Their testimonials “add context to the data and add complexity that is oftentimes just not seen in raw numbers”, says Wilson. Although he has served on many committees, he says that the report is the first document he’s seen that includes these types of interviews. “It was a learning experience for me,” he says. “I think it’s a very powerful tool.”The committee began meeting before a wave of legislation by some US states seeking to clamp down on diversity initiatives and the inclusion in university curricula of the topic of racism. In addition, reports show that although the number of DEI roles created to help companies and institutions achieve a balanced workforce increased by 50% following the murder of George Floyd — a Black man who was killed in an encounter with police in Minneapolis, Minnesota in 2020 — the attrition rate of those roles had risen to as high as 33% by the end of last year. Abby Ray, vice-president of marketing and communications at oSTEM, a national non-profit organization based in Grandville, Michigan, that advocates for people from sexual and gender minorities in science, says that such efforts underscore the need to keep pushing for change. “It is essential,” they say, “that we continue to uplift those individuals who have historically been oppressed and deliberately excluded from STEMM fields.”\\n',\n",
       "  'img_link': 'https://media.nature.com/lw767/magazine-assets/d41586-023-00823-6/d41586-023-00823-6_24607590.jpg'},\n",
       " {'title': 'Tormentor mentors, and how to survive them',\n",
       "  'summary': '<p>Nature, Published online: 16 March 2023; <a href=\"https://www.nature.com/articles/d41586-023-00821-8\">doi:10.1038/d41586-023-00821-8</a></p>Bad mentors can go absent, sap your energy or embroil you in their paranoia. Here are five tips for tackling a toxic relationship.',\n",
       "  'field': 'careers,lab life,research management',\n",
       "  'content': '\\n\\n\\n\\n\\n\\n\\nA mentor is the guide by your side who can help to develop your career, and cheer you on as you succeed and encourage you when you fail. But what happens when your mentor becomes uncooperative or resistant, or completely ghosts you by suddenly ending communication? Your mentor has now become a tormentor.In a laboratory environment, the principal investigator is often a ‘mentor by default’ — which comes with the responsibilities of helping the trainee to develop professionally and paving a pathway for their future endeavours. Other mentors might include faculty members, postdoctoral colleagues and managers.The repercussions of poor mentorship are far-reaching, from delayed career advancement to a trainee leaving science altogether.We have both written about scientific careers, researched the world of scientific mentoring and developed mentoring programmes. We’ve also both been lucky enough to be mentors, and have helped others to escape bad mentoring relationships. We have some advice on identifying and recognizing poor mentorship.The examples below illustrate poor mentorship in the workplace, although there are more-extreme cases, such as verbal or physical abuse. In cases of abuse or exploitation, we recommend seeking help directly from the appropriate institutional authority, such as campus police, your human-resources department or any kind of mediator.Harmful behaviours can damage a trainee’s career, or even their health. Here’s some advice on how you can identify and address them.Common tormentor behavioursThe ghost mentor. This mentor has an unreasonably long lag time for offering feedback on time-sensitive issues, such as manuscripts and presentations. They fail to respond to follow-up e-mails, preventing the trainee from pushing forwards in their career. The mentor might stall projects, set unrealistic goals or create unnecessary barriers to progress. This could cause a delay in the trainee’s academic timeline and risks a multitude of setbacks to their professional goals.The elitist mentor. This mentor chooses to submit papers only to high-impact journals, lowering their trainee’s chances of publication. This could delay or completely prevent a publication — therefore limiting the trainee’s future career opportunities.The paranoid mentor. This mentor forbids or strongly advocates against public presentations at conferences or other collaboration proposals, out of fear of their own work being scooped. This could cause their trainee to miss out on building valuable science-communication skills or connecting with potential collaborators. As science becomes more team-oriented, this behaviour becomes more and more damaging.\\n\\nSupport letters: mostly ghost-written, always glowing. What’s the point?\\nThe dementor mentor. This type of mentor, much like the happiness-sapping creatures in the Harry Potter books that they’re named after, withholds support or encouragement when their trainee is facing scientific hardship, such as a rejected paper, failed experiment or poor grant score. Dementor mentors might leave the trainee to navigate projects alone, despite many needing more hands-on guidance. Just when the trainee needs a cheerleader and an alternative, authoritative perspective, dementors lack the emotional intelligence to offer much-needed support.These behaviours can affect trainees’ psychosocial needs, contributing to feelings of unimportance in the lab environment, a decreased sense of self-efficacy, increased imposter syndrome and burnout. This goes far beyond ghosting, because the mentor fails to provide pivotal emotional support.How to deal with a tormentorA tormentor might be a barrier to success, but trainees do not need to continue an unhappy mentoring relationship. There are steps they can take to draw a line under past misunderstandings and move on, or widen their mentoring network to include other perspectives. As a last resort, they could move labs.Often, the power dynamic favours the mentor, and the trainee feels powerless, especially if the mentor has direct line-management responsibility or runs the lab. If you are a trainee in a bad mentoring relationship, here are five steps you can take to circumvent a tormentor and take back your power.Interpersonal relationships are complicated, and this advice is not a silver bullet that will solve all mentorship problems, but it will help you to identify toxic behaviours. Only you can decide whether a behaviour is acute or chronic, and just how much you can and should handle. In a bad mentorship relationship, you might feel depleted, overwhelmed or isolated — and knowing that the management style and mentoring you are receiving might fall outside acceptable norms will help to validate your feelings. You can then decide on a course of action.Name your next goalInstead of working towards an undefined finishing line, clearly articulate your next professional goal, both to yourself and to your mentor. For example, if you’re a PhD student, then perhaps defending your thesis is your next major milestone. Naming your goal will help you to align your activities, manage your time and know what guidance to seek from your mentor.\\n\\nMentoring resources\\nStart developing a planSetting deadlines brings goals to life. Come to a mentoring meeting with a clear goal and plan. Ask your mentor to help you refine the plan and develop a timeline, and whether there are any challenges you should be aware of. You can also ask them to refer you to others for extra guidance or support. For example, if your goal is to defend your thesis, what are the next logical steps? Perhaps you need to have a first-author manuscript or finish a series of experiments.Lean on your mentoring teamWhen you are in the middle of a chaotic situation, you could have blind spots that might cause you to miss areas of potential conflict. It is difficult to see both the big picture and nuances when you’re deeply rooted in these situations. It’s always advisable to have more than one mentor, and surrounding yourself with a diverse team can offer fresh perspectives on challenging situations — and, at times, your mentors might advocate on your behalf.Examine professional dynamicsNot every setback is caused by a tormentor mentor. Is your mentor being appropriately tough or creating legitimate barriers? Consider whether there are more factors at play than you had initially taken into account, and how to address them. For example, perhaps the project is not as strong as it could be, and needs extra work to support the final product. However, being asked to do what you consider to be unnecessary work could be a sign of the mentor’s insecurities about the project, or about you as their trainee. It might be time to start keeping a paper trail documenting obstructions if you feel that your mentor is holding you back. Consult with other mentors, if you have them, or with your thesis committee.Re-evaluate optionsIf you’ve gone through all the other steps and are still experiencing insurmountable roadblocks, it might be a good time to consider alternative options, such as a co-mentor, a new lab or a leave of absence while you re-evaluate your future without daily toxic interference. Feeling frustrated, upset and disappointed is natural as you grieve for the lab experience you had expected. Change is hard under any circumstance; doing it under duress is particularly challenging and overwhelming.Your newly refined professional goals can inform your path forwards. Use the lessons from your experience with a tormentor as a guide to what to avoid in future mentoring dynamics. When looking for a new mentor, interview both the mentor and their current trainees on how the mentor will support your goals.Bad mentoring is worse than no mentoring. However, a toxic mentorship does not need to be a life sentence. There are opportunities to take action, meet your goals and shed your tormentor.\\n',\n",
       "  'img_link': 'https://media.nature.com/lw767/magazine-assets/d41586-023-00821-8/d41586-023-00821-8_24621598.jpg'},\n",
       " {'title': 'How do we smell? First 3D structure of human odour receptor offers clues',\n",
       "  'summary': '<p>Nature, Published online: 16 March 2023; <a href=\"https://www.nature.com/articles/d41586-023-00818-3\">doi:10.1038/d41586-023-00818-3</a></p>Finding could advance our understanding of how human olfactory proteins recognize specific scents, including the pong of ripe cheese.',\n",
       "  'field': 'molecular biology',\n",
       "  'content': '\\n\\n\\n\\n\\n\\n\\nIt is thanks to proteins in the nose called odour receptors that we find the smell of roses pleasant and that of rotting food foul. But little is known about how these receptors detect molecules and translate them into scents.Now, for the first time, researchers have mapped the precise 3D structure of a human odour receptor, taking a step forwards in understanding the most enigmatic of our senses.The study, published in Nature on 15 March1, describes an olfactory receptor called OR51E2 and shows how it ‘recognizes’ the smell of cheese through specific molecular interactions that switch the receptor on.“It’s basically our first picture of any odour molecule interacting with one of our odour receptors,” says study co-author Aashish Manglik, a pharmaceutical chemist at the University of California, San Francisco.Smell mysteryThe human genome contains genes encoding 400 olfactory receptors that can detect many odours. Mammalian odour-receptor genes were first discovered in rats by molecular biologist Richard Axel and biologist Linda Buck in 19912. Researchers in the 1920s estimated that the human nose could discern around 10,000 smells3, but a 2014 study suggests that we can distinguish more than one trillion scents4.Each olfactory receptor can interact with only a subset of smelly molecules called odorants — and a single odorant can activate multiple receptors. It is “like hitting a chord on a piano”, says Manglik. “Instead of hitting a single note, it’s a combination of keys that are hit that gives rise to the perception of a distinct odour.”Beyond this, little is known about exactly how olfactory receptors recognize specific odorants and encode different smells in the brain.Technical challenges in producing mammalian olfactory-receptor proteins using standard laboratory methods have made it difficult to study how these receptors bind to odorants.“Almost all of them really don’t like being in any other kind of cell other than an olfactory sensory neuron,” says Matthew Grubb, a neuroscientist at King’s College London. This means that they cannot be grown or stabilized in commonly used cell lines. “You would have to dissect probably thousands of mice noses” to replicate samples, says Grubb. “It’s just not feasible.”To overcome this, Manglik and his colleagues focused on the OR51E2 receptor, which has functions beyond odorant recognition and is found in gut, kidney and prostate tissues, as well as olfactory neurons.Vinegar and cheeseOR51E2 interacts with two odorant molecules: acetate, which smells like vinegar, and propionate, which has a cheesy odour.The authors purified the receptor and analysed the structure of propionate-bound and unbound OR51E2 using cryo-electron microscopy, an atomic-resolution imaging technique. They also used computer-aided simulations to model how the protein interacts with the odorant at an atomic scale.They found that propionate binds OR51E2 through specific ionic and hydrogen bonds that anchor the propionate’s carboxylic acid to an amino acid, arginine, in a region of the receptor called the binding pocket. Binding to propionate alters the shape of OR51E2, which is what turns the receptor on.These molecular interactions are crucial: the researchers showed that mutations affecting arginine prevented OR51E2 from being activated by propionate.“This is our way of kind of lining up the dominoes to understand how pushing on one side of the receptor turns the other side on,” says Manglik.On the scentScientists have long dreamed of building a molecular atlas of olfactory receptors that maps their chemical structures and which combinations of receptors correspond to particular odours. But “that’s been very much out of reach for the field”, says Manglik.The OR51E2 receptor is specific to propionate and acetate. But “it’s not all about single odorant binding to single receptor molecules”, says Grubb. OR51E2 is a class I olfactory receptor; only around 10% of human olfactory-receptor genes encode this type. The rest code for class II receptors, which typically recognize a broader range of odours. “They may have very different mechanisms,” says Vanessa Ruta, a neuroscientist at the Rockefeller University in New York City.Studying other examples of human odour receptors and elucidating their structures is crucial, she adds. “It will allow for a broader understanding of the different ways that odorants are recognized.”\\n',\n",
       "  'img_link': 'https://media.nature.com/lw767/magazine-assets/d41586-023-00818-3/d41586-023-00818-3_24631076.jpg'},\n",
       " {'title': '‘Spell-checker for statistics’ reduces errors in the psychology literature',\n",
       "  'summary': '<p>Nature, Published online: 15 March 2023; <a href=\"https://www.nature.com/articles/d41586-023-00788-6\">doi:10.1038/d41586-023-00788-6</a></p>Developed to detect statistical errors, statcheck reduces mistakes in reported P values by up to 4.5-fold.',\n",
       "  'field': 'psychology,software,publishing,peer review',\n",
       "  'content': '\\n\\n\\n\\n\\n\\n\\nA free-to-use tool designed to detect statistical errors is significantly reducing the number of mistakes creeping into papers, suggests a study published this January1.First described in a 2015 publication2, statcheck is an online tool that can identify errors in the P values — a controversial but popular statistical measure — reported in scientific papers. The tool, developed by Michèle Nuijten, a methodologist at Tilburg University in the Netherlands, has been applied to tens of thousands of psychology studies during peer review and by authors themselves, and according to a 2017 study, it is right more than 95% of the time3.Several journals have now implemented statcheck — which Nuijten calls a spell-checker for statistics — in their peer-review processes. In the new study, which was published on the PsyArXiv preprint server and has not yet been peer-reviewed1, Nuijten set out to quantify the tool’s impact by comparing the rate of errors in P values in two psychology journals before and after they implemented statcheck, and in two journals that have never used the tool.Error reductionThe team looked at more than 7,000 papers published since 2003, and found that papers published in Psychological Science and Journal of Experimental and Social Psychology had 4.5 percentage points fewer statistical errors after they started using statcheck. By contrast, the Journal of Experimental Psychology: General and Journal of Personality and Social Psychology, which don’t use the tool, had a reduction of only around one percentage point.\\n\\nNatureTech hub\\nThat means the journals using statcheck had a more than fourfold reduction in errors in reported P values compared with those that have not implemented it, Nuijten says.Nuijten attributes the reduced error rate in papers in the journals that don’t use statcheck to an increased awareness of issues around statistical significance over the study period, as well as to researchers using statcheck themselves before they submit their manuscripts.The software is available as a web application to which researchers can upload their manuscripts. It has been accessed more than 100,000 times since its launch in September 2016. The tool is also available as a plug-in for Microsoft Word to help researchers identify errors during the writing process.Patricia Bauer, a psychologist at Emory University in Atlanta, Georgia, and editor-in-chief of Psychological Science, says, “statcheck is a free, easy to use, easy to implement step that authors can take and that the journal can require.” The journal requires authors to submit a clean statcheck report with no reported errors as a prerequisite to publication.But Anna-Lena Schubert, a psychologist at the Johannes Gutenberg University of Mainz in Germany and secretary of the German Psychological Society, worries that it might not be possible to obtain a clean statcheck report if the tool has made mistakes. In those cases, she says, authors should have the opportunity to submit a rebuttal.Still, Schubert concludes, “given its relatively low error rate, it would be very beneficial for psychology journals to adopt statcheck as a screening tool to increase the rigour and robustness of psychology literature”.\\n',\n",
       "  'img_link': 'https://media.nature.com/lw767/magazine-assets/d41586-023-00788-6/d41586-023-00788-6_24084972.jpg'},\n",
       " {'title': 'UN high seas treaty is a landmark – but science needs to fill the gaps',\n",
       "  'summary': '<p>Nature, Published online: 15 March 2023; <a href=\"https://www.nature.com/articles/d41586-023-00757-z\">doi:10.1038/d41586-023-00757-z</a></p>The agreement is a once-in-a-generation opportunity for researchers and funders to use every idea and instrument available to preserve the health of the seas.',\n",
       "  'field': 'policy,biodiversity,ocean sciences,environmental sciences,climate change',\n",
       "  'content': '\\n\\n\\n\\n\\n\\n\\nThe United Nations high seas treaty has been a long time coming. Secured earlier this month after almost 20 years of effort, it will be the first international law to offer some protection to the nearly two-thirds of the ocean that is beyond national control. These parts of the ocean currently have few, if any, meaningful safeguards against pollution, overfishing and habitat destruction. The treaty is without doubt a major achievement.Agreed under the UN Convention on the Law of the Sea, it represents several wins. Among them is the capacity to create marine protected areas through decisions of a conference of the parties to the treaty. It also recognizes that genetic resources of the high seas must benefit all of humanity. Moreover, companies planning commercial activities and organizations considering other large projects (such as potential climate interventions involving the ocean) will need to carry out environmental impact assessments.\\n\\nUN forges historic deal to protect ocean life: what researchers think\\nCountries will be permitted to profit from exploiting marine genetic resources, but they must channel a proportion of their profits into a global fund to protect the high seas. Although the details are still to be worked out, high-income countries active in marine genetic research will be asked to contribute proportionately more to the fund.The treaty contains many opportunities for research in ocean science, for building research capacity in low- and middle-income countries, and for improving the evidence available to decision makers. Researchers working with marine genetic resources will need to register their interests with a central clearing house and commit to making data and research outputs open access.Scientists will have an important role in ensuring the treaty’s ultimate success. In part, this will involve gathering or improving the evidence to support the establishment and maintenance of strong marine protected areas and to inform stringent environmental impact assessments. Beyond that, researchers must make every effort to ensure transparency, including declaring the origin and prospective use of any genetic material, and making digital sequence information available through international repositories. This will not only enhance cooperation and capacity-building, but will also help governments to develop their own national regulations and procedures in line with the treaty.There’s also the potential for fresh scientific collaboration — for example, using emerging technologies such as telepresence, whereby scientists can take part in research cruises remotely. Marine scientists travelling to, say, the Pacific Ocean could collect samples under the guidance of colleagues elsewhere in real time. The knowledge gained from such collaborations could lead to the commercialization of new products, benefiting scientists and economies around the world.However, it is important not to overstate the treaty’s potential: notwithstanding its successes, there are deficiencies that the international community, supported by the research community, must now work to remedy.\\n\\n\\n\\n\\nRena Lee, president of the high seas treaty conference, concluded proceedings on 3 March with the words “the ship has reached the shore”.Credit: Kena Betancur/AFP/Getty\\n\\n\\nAs the planet warms, the Arctic’s permanent ice cover is melting, and China is planning a shipping route through the Central Arctic Ocean. This could become a regular passageway for shipping between Asia and Europe within a decade. In the Pacific, mining companies are exploring the deep sea bed for metals that they say are needed for the batteries that will power the coming green-energy transition. But these activities won’t face scrutiny under the treaty, because the treaty’s provisions don’t overrule regulations laid down by the authorities that oversee existing high seas activities. These include the International Maritime Organization, which is responsible for shipping; the International Seabed Authority, which oversees deep-sea mining; and some 17 regional fisheries management organizations tasked with regulating fisheries in various parts of the ocean, including Antarctica. Military activities and existing fishing and commercial shipping are, in fact, exempt from the treaty.\\n\\nProtecting the ocean requires better progress metrics\\nThis means, for example, that the treaty cannot create protected areas in places already covered by fishing agreements, even if that fishing is unsustainable and depleting stocks. This is a gaping hole. The overexploitation of coastal fisheries has made a frontier of the high seas, as fleets travel farther and fish for longer in search of dwindling resources. One outcome is that stocks of some highly migratory species, such as tuna, have dropped precipitously since the 1950s (M. J. Juan-Jordá et al. Proc. Natl Acad. Sci. USA 108, 20650–20655; 2011). By 2020, the Pacific bluefin tuna, for instance, was at 10.2% of the levels before fishing began (see go.nature.com/3n2q5th). Oceanic sharks and rays have also declined globally by 71% since 1970 (N. Pacoureau et al. Nature 589, 567–571; 2021). Once the treaty becomes law (after it has been ratified in the national parliaments of at least 60 countries), it can demand that proposed ocean activities — such as climate-intervention experiments — are subject to stringent environmental impact assessments. But it cannot do the same for activities already under way.Nor will the treaty end current offshore environmental violations. Farming waste, in the form of excessive nutrients, routinely ends up in rivers and coastal waters. From there, it makes its way to the open ocean, where it results in the formation of dead zones — vast areas devoid of life. Between 2008 and 2019, the number of these zones nearly doubled, from 400 to 700 (see go.nature.com/3mpigh1). So much plastic is now entering our seas that the oceans are thought to contain around 200 million tonnes. Meanwhile, cruise ships legally discharge more than one billion tonnes of raw sewage into international waters every year.Nonetheless, as humanity’s first serious attempt to challenge the carnage that prevails offshore, the high seas treaty is a triumph for diplomacy, particularly at a time when multilateralism is under sustained pressure. At present, just 1% of international waters are protected. That proportion is now set to grow, and this will help to maintain the health of our oceans and stem biodiversity loss. In securing this deal, the international community has given itself a fighting chance of coming good on earlier promises — most recently reiterated under the UN Convention on Biological Diversity — to protect 30% of the ocean by 2030.Full implementation, although some years away, offers scientists a once-in-a-generation opportunity to use their knowledge to support offshore conservation. In redressing our ‘out of sight, out of mind’ relationship with the oceans, the high seas treaty will allow us — supported by a burgeoning research effort — to rethink how we use our ocean commons in ways that benefit the majority.\\n',\n",
       "  'img_link': 'https://media.nature.com/lw767/magazine-assets/d41586-023-00757-z/d41586-023-00757-z_24621492.jpg'},\n",
       " {'title': 'Volcanoes on Venus? ‘Striking’ finding hints at modern-day activity',\n",
       "  'summary': '<p>Nature, Published online: 15 March 2023; <a href=\"https://www.nature.com/articles/d41586-023-00783-x\">doi:10.1038/d41586-023-00783-x</a></p>Discovery highlights need for future missions after NASA puts one on hold.',\n",
       "  'field': 'planetary science,volcanology,astronomical instrumentation',\n",
       "  'content': '\\n\\n\\n\\n\\n\\n\\nScientists have found some of the strongest evidence yet that there is volcanic activity on Venus. Because the planet is a close neighbour to Earth and originally had water on its surface, one big question has been why its landscape is now hellish while Earth’s is habitable. Learning more about its volcanic activity could help explain its evolution — and Earth’s.\\n\\nHow three missions to Venus could solve the planet’s biggest mysteries\\nScientists have known for some time that Venus is covered in volcanoes, but whether any of them are still active has been long debated. By examining radar images of the planet’s surface collected by NASA’s Magellan spacecraft between 1990 and 1992, researchers have determined that a volcanic vent located in Venus’s Atla Regio area, which contains two of the planet’s largest volcanoes, changed shape between two images taken eight months apart, suggesting an eruption or flow of magma beneath the vent. The scientists reported their findings on 15 March in Science1 and presented them at The Lunar and Planetary Science Conference in the Woodlands, Texas, on the same day.This is a “striking find”, says Darby Dyar, an astronomer at Mount Holyoke College in South Hadley, Massachusetts. It brings the astronomy community one step closer to figuring out how Venus works, adds Dyar, who is also deputy principal investigator of the VERITAS mission to Venus, which is being overseen by NASA’s Jet Propulsion Laboratory (JPL) in Pasadena, California, and aims to map the planet’s surface sometime after 2030. “The whole subject of whether there is active volcanism on the surface of Venus suffers from a lack of data.”A hellish planetGathering evidence that the planet is volcanically active wasn’t easy. Venus’s thick atmosphere — 100 times the mass of Earth’s — and high temperatures of 450 ºC make it difficult for rovers and other probes to explore the surface. So far, the most reliable data scientists have collected have come from the Magellan spacecraft.Robert Herrick, a geophysicist at the University of Alaska Fairbanks, and Scott Hensley, a radar scientist at JPL who is also part of the VERITAS team, analysed full-resolution radar images captured by Magellan of areas with suspected volcanic activity.\\n\\nVenus is Earth’s evil twin — and space agencies can no longer resist its pull\\nThe challenge was that Magellan imaged the planet in three cycles over the 24-month period analysed by the scientists. During each cycle, the spacecraft pointed its radar to Venus’s surface at a different angle. For the scientists to look for changes on the surface over time, they had to superimpose the images at various angles and find overlaps in the terrain to line them up.Herrick compares the problem to flying from multiple directions through the Grand Canyon in Arizona and then trying to map its surface while looking at opposite canyon walls. “Trying to find the same things in those images gets a little more challenging,” he says.The low resolution of the Magellan images added another layer of complexity. “You’re looking at the surface, where a football field is a single pixel,” he adds.This worries Scott King, a geophysicist at Virginia Tech in Blacksburg who studies Venus. He questions whether the images are strong enough evidence to convince sceptics that Venus is volcanically active. “Proof is in the eye of the beholder,” he says.Herrick and Hensley acknowledge this limitation in their data. But they also say they are not aware of any equivalent events on Earth that could cause the observed changes without volcanic activity, although they cannot rule out the possibility that something else might have been responsible.King doesn’t find it hard to believe that the planet has volcanic activity. He hopes, however, that upcoming missions to Venus, including VERITAS, will provide the data needed to convince everyone.An overlooked planetVERITAS has been delayed, however — so King might be waiting longer than originally thought. NASA had planned to launch the mission in 2028, but the agency had to reallocate JPL’s funding to address the delay of Psyche, a mission that will study a metal-rich asteroid orbiting the Sun between Mars and Jupiter. NASA does not currently have funds planned for VERITAS, and even if it restores funding, the mission would launch no earlier than 2031.\\n\\nProspects for life on Venus fade — but aren’t dead yet\\nLaunching VERITAS after 2030 could cause problems for other missions, Dyar says. Ideally, the topographic data collected by VERITAS would have provided NASA’s DAVINCI and the European Space Agency’s EnVision missions with information to help them target the areas they’re planning to explore. DAVINCI, set to launch in 2029, aims to drop a probe into Venus’s atmosphere, and EnVision, set to launch in the early 2030s, is meant to take high-resolution radar images of the planet’s surface.Studying Venus could not only help researchers understand more about how Earth works, but also help them learn more about exoplanets beyond the Solar System. “We’re discovering hundreds, thousands of exoplanets,” Dyar says. And many of those seem to be Venus-like, she adds.Many space missions have been targeting Mars recently, even though Venus is much more Earth-like than the red planet is overall. Herrick hopes that the latest findings will motivate people to turn their eyes towards Venus and prompt NASA to launch VERITAS on time. “Venus is truly Earth’s sibling,” he says.\\n',\n",
       "  'img_link': 'https://media.nature.com/lw767/magazine-assets/d41586-023-00783-x/d41586-023-00783-x_24626702.jpg'},\n",
       " {'title': 'Common sweetener suppresses mouse immune system — in high doses',\n",
       "  'summary': '<p>Nature, Published online: 15 March 2023; <a href=\"https://www.nature.com/articles/d41586-023-00784-w\">doi:10.1038/d41586-023-00784-w</a></p>Finding suggests that the sugar substitute sucralose could one day be used to treat autoimmune conditions.',\n",
       "  'field': 'diseases,nutrition,immunology',\n",
       "  'content': '\\n\\n\\n\\n\\n\\n\\nHigh doses of sucralose — a potent, calorie-free sugar substitute that is 600 times sweeter than sucrose — reduce immune responses in mice, a study has found.The researchers did not investigate the sweetener’s effects in humans, and say that it is unlikely that normal consumption of sucralose is harmful. But the results, published on 15 March in Nature1, suggest that the sweetener has a clear biological effect beyond stimulating taste.\\n\\nSugar substitutes linked to obesity\\n“There has been this world view that these sweeteners would just wash through our bodies — our tongues would taste them and nothing else would happen,” says Susie Swithers, a behavioural neuroscientist at Purdue University in West Lafayette, Indiana, who studies the health effects of using artificial sweeteners and was not involved with the research. “This study is yet another piece of evidence that that’s profoundly untrue.”Although the authors call for more research to better understand any impacts of the molecule on people’s health, they also suggest that it could be used to tamp down conditions that cause a hyperactive immune system.“What an impressive work,” says Guillaume Walther, a physiologist at the University of Avignon in France who studies the health effects of sucralose. “The rigour of the study and experiments in this paper is incredible.”Immune impairmentArtificial sweeteners have come under scrutiny over the past few decades, because researchers have found that some sugar substitutes have biological effects, such as altering people’s gut microbes2.To investigate whether sucralose also has an effect on the immune system, Fabio Zani, a molecular biologist at the Francis Crick Institute in London and his colleagues carried out laboratory tests in which immune cells called T cells, taken from mice and humans, were exposed to the sweetener. They found that sucralose impaired the T cells’ ability to replicate and specialize.To see whether the effect was the same in living animals, the researchers gave mice water bottles containing a dose of sucralose that is the rodent equivalent of the maximum safe intake in humans — a standard set by regulatory authorities such as the US Food and Drug Administration. The mice had either a bacterial infection or a tumour, which allowed the team to see how responsive their immune systems were. These mice showed impaired T cell responses, compared with mice in control groups that were given either water or other sweeteners. When the team stopped giving the mice sucralose, their T cell responses began to recover.\\n\\nHealthy people, healthy planet: the search for a sustainable global diet\\nThe researchers didn’t test lower doses of sucralose, but it “seemed very clear to us that if we went much lower, we probably would have lost the effect altogether”, says study co-author Karen Vousden, a cancer biologist at the Francis Crick Institute. “We’re pretty confident that the amount that people take in their normal diet is not going to have any effect.”The sweetener seems to impair only T cells, and not other immune cells, such as B cells or myeloid cells, and it doesn’t accumulate inside the T cells. Previous research has shown that sucralose can affect the fluidity of cell membranes, which might make it more difficult for T cells to communicate, the authors speculate.Robert Rankin, executive director of the Calorie Control Council, a trade group representing companies that produce reduced-calorie food and beverages, notes that the study focused on mice and that the doses exceed the amount that people would typically consume.Sweet medicineSucralose’s effects on the immune system aren’t inherently negative, Vousden says. The results highlight the possibility that the sweetener could one day be used therapeutically to treat autoimmune conditions, she says.To test this theory in animals, the researchers gave high doses of the sweetener to mice that were bred to be predisposed to type 1 diabetes, an autoimmune condition that causes T cells to attack pancreatic cells. After about 30 weeks, only about one-third of the mice that were given the sweetener developed diabetes; by contrast, all the mice that were given only water had developed the condition.Zani says that if future research were to find a similar effect in humans, he could see the sweetener being administered alongside more-conventional immunosuppressive drugs. This could allow physicians to lower the dosages of these drugs. This avenue of research is promising, says Walther, especially because sucralose is cheap to manufacture and would have fewer unwanted side effects.\\n',\n",
       "  'img_link': 'https://media.nature.com/lw767/magazine-assets/d41586-023-00784-w/d41586-023-00784-w_24626600.jpg'},\n",
       " {'title': 'Fastest-ever calcium sensors broaden the potential of neuronal imaging',\n",
       "  'summary': '<p>Nature, Published online: 15 March 2023; <a href=\"https://www.nature.com/articles/d41586-023-00704-y\">doi:10.1038/d41586-023-00704-y</a></p>Proteins have been developed that emit flashes of light in response to influxes of calcium ions into cells on millisecond timescales. Two sets of scientists discuss the legacy and future of these proteins.',\n",
       "  'field': 'neuroscience,biotechnology',\n",
       "  'content': '\\nTHE PAPER IN BRIEF• Genetically encoded calcium indicators (GECIs) are designer proteins that react to intracellular calcium-ion (Ca2+)influxes — which occur during the activation of neurons — by emitting light.• GECIs have enabled researchers to gain myriad insights into the workings of the brain, but until now the proteins have lacked the precision to record a fundamental unit of activity, known as the action potential, in single neurons.• Writing in Nature, Zhang et al.1 describe a GECI that marks a step towards solving this problem.MICHAEL B. RYAN & ANNE K. CHURCHLAND: Lamps that illuminated the brainImagine rowing across still, dark waters on a summer night, when the water is suddenly transformed by the brilliant glow of countless jellyfish illuminating a hidden world of activity. In the 1960s, scientists from the Friday Harbor Laboratories in Washington collected nearly 10,000 specimens of Aequorea jellyfish by hand, with the goal of isolating the organic component responsible for this natural bioluminescence2. The scientists extracted and purified a calcium-sensitive protein that they named aequorin, together with an associated green fluorescent protein (GFP). Soon after, researchers loaded purified aequorin into living cells to track changes in Ca2+, an ion that is central to neuronal activity3. This ability to measure neuronal activity by proxy — through flashes of light — revolutionized the field of neuroscience, and led to the identification of previously unknown dynamics in the brain.By injecting aequorin into neurons, scientists discovered how changes in the levels of intracellular Ca2+, resulting from neuronal activation, contribute to the process by which these cells release neurotransmitters to communicate with one another4. Several decades after aequorin was first isolated, another breakthrough came in the form of GECIs. Whereas aequorin and other calcium-sensitive dyes require manual loading into cells, GECIs are engineered to become part of the organism’s DNA5,6. The most widely used GECIs, known as the GCaMP family7, consist of a calcium-binding protein called calmodulin and a bound peptide (a short string of amino acids), fused to a modified form of GFP. When Ca2+ binds to calmodulin, this triggers a conformational change in the fusion protein, which causes GFP to emit light (Fig. 1a).\\n\\n\\n\\n\\n\\nDespite the potential of these calcium sensors, early versions of GCaMP were hampered by issues such as poor response kinetics, low calcium sensitivity and limited correlation with the electrical activity of neurons. Initial versions of the sensors could detect only large changes in Ca2+ brought about by tens of action potentials, with response kinetics on the order of several hundred milliseconds.Considerable effort has since been dedicated to improving the performance of GCaMP sensors through structure-guided design. Later versions of GCaMP have even been optimized for sensitivity or speed, with ‘slow’ variants maximizing signal strength, and ‘fast’ variants maximizing response kinetics. Fast variants of GCaMP can now detect changes in calcium in time frames of tens of milliseconds. Other GECIs have also been developed that emit different wavelengths of light in response to Ca2+, enabling several populations of neurons to be imaged simultaneously.\\n\\nRead the paper: Fast and sensitive GCaMP calcium indicators for imaging neural populations\\nThese advances have enabled scientists to capitalize on the versatility, specificity and longevity of GECIs. The fact that they are genetically encoded allows them to be expressed continuously in many cells at once. This has facilitated in vivo measurements of large populations of cells in flies, rodents and primates. The cell type in which GECIs are expressed can also be controlled genetically, which has helped researchers to understand the variability of neural circuits. For instance, GECIs have been used to delineate subpopulations of neurons in the brain’s cortex that have distinct calcium dynamics and roles in decision-making8. They have also been used to measure Ca2+ in non-neuronal cells in the brain. For example, calcium imaging of astrocytes — the most abundant non-neuronal cell type in the brain — has revealed how neurotransmitters such as dopamine shape the response of astrocytes to neural activity9,10.Nonetheless, the latest GCaMP iterations have fallen short of reliably identifying individual action potentials under the ‘noisy’ conditions in living brains. Several research groups have been racing to identify GCaMP variants that can detect these changes — which occur over just a few milliseconds — without sacrificing brightness. Zhang and colleagues’ work introduces a competitor to this race, and points to a bright future for calcium imaging.YIYANG GONG & CASEY BAKER: Time for the next generationSingle action potentials are the fundamental unit of neural communication. Zhang and colleagues’ development is therefore a major advance. The authors’ GCaMP8 sensors have significantly faster kinetics than do existing start-of-the-art GECIs (Fig. 1b), such as the GCaMP7 and XCaMP series11,12. The GCaMP8 sensors also avoid the typical trade-off between sensitivity and speed, whereby slower sensor kinetics produce larger optical responses.Zhang and colleagues based their sensors on GCaMP6. The authors optimized this protein by engineering different versions of several of its modules, including the two linkers that connect GFP to the bound peptide and to the calmodulin domain that grabs the peptide when Ca2+ binds. They paid particular attention to the interface between the bound peptide (which in GCaMP8 is a fragment of endothelial nitric oxide synthase) and the calmodulin domain, because this region of the sensor has a crucial role in its response and kinetics12,13. The team then used an optimization process that involved replacing many amino acids and performing several rounds of testing to identify the best-performing sensors. They validated the performance of these indicators in flies and mice. The strong performance of the GCaMP8 sensors suggests that the way in which Ca2+ induces fluorescence in this series is different from that in other GCaMPs.\\n\\nA breakthrough method that became vital to neuroscience\\nThe fast kinetics and high accuracy of the GCaMP8 sensors will enable researchers to analyse phenomena in live animals that could previously have been studied only by using electrical measurements or genetically encoded voltage indicators (GEVIs), which respond to changes in voltage by fluorescing (much like how GECIs respond to calcium). Until now, it has been assumed that GECIs and GEVIs report different types of neural activity — GEVIs determine the timing of action potentials, and GECIs reveal key activity in neuronal compartments (for example, by measuring Ca2+ dynamics in processes called dendrites that receive signals from other neurons)14.However, with the development of GCaMP8, the information that can be gained from these tools has converged. Zhang and colleagues show, through computational analyses of the light bursts released by GCaMP8, that their sensors can detect action potentials nearly as accurately as can voltage imaging15,16. Similarly, GCaMP8 could reliably detect when fly neurons that normally fire frequently become transiently inactive, providing similar results to GEVI-based measurements17,18.When both voltage and calcium imaging can be used to record from many neurons simultaneously, practical experimental considerations might persuade future researchers to choose calcium imaging over voltage imaging. GCaMP8 is compatible with existing microscopy configurations and preparation methods. The series can be used to examine the activity of broad neuronal networks on a larger scale than is possible using voltage imaging. One might imagine that the GCaMP8 series will soon be used to interpret detailed, millisecond-by-millisecond sequences of activity that flow through many neurons in a specific area of the brain.The fast kinetics associated with this generation of GCaMPs, along with existing GEVIs, will motivate the development of rapid optical-microscopy and imaging technologies. Such microscopes would be able to detect transient bursts of light emitted from many neurons simultaneously. But for now, the fact that calcium-sensor kinetics is no longer the bottleneck in interpreting rapid neural activity will be welcomed by many neuroscientists.\\n',\n",
       "  'img_link': 'https://media.nature.com/lw767/magazine-assets/d41586-023-00704-y/d41586-023-00704-y_24616222.png'}]"
      ]
     },
     "execution_count": 80,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "id": "1e6af20b",
   "metadata": {},
   "outputs": [],
   "source": [
    "r = requests.get(\"https://www.nature.com/articles/d41586-023-00795-7\")\n",
    "soup = BeautifulSoup(r.content, \"html.parser\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "id": "412843b7",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['Careers', 'Physics', 'Nanoscience and technology']"
      ]
     },
     "execution_count": 9,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "soup.find(\"ul\",{\"class\":\"c-article-subjects u-clearfix u-hide-print\"}).text.strip().split(\"\\n\\n\\n\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 42,
   "id": "798d7060",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'//media.nature.com/w767/magazine-assets/d41586-023-00795-7/d41586-023-00795-7_24630594.jpg'"
      ]
     },
     "execution_count": 42,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "soup.find(\"figure\",{\"class\":\"figure\"}).img[\"src\"]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "id": "b706430c",
   "metadata": {},
   "outputs": [],
   "source": [
    "_test_ls = [\n",
    "    \"https://www.nature.com/articles/d41586-023-00831-6\",\n",
    "    \"https://www.nature.com/articles/d41586-023-00818-3\",\n",
    "    \"https://www.nature.com/articles/d41586-023-00816-5\"\n",
    "]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 100,
   "id": "74e62ee4",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'//media.nature.com/lw767/magazine-assets/d41586-023-00831-6/d41586-023-00831-6_24636118.jpg'"
      ]
     },
     "execution_count": 100,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "soup.find(\"img\",{\"class\":\"figure__image\"})[\"src\"]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "id": "9def1a88",
   "metadata": {},
   "outputs": [],
   "source": [
    "lorem = \"\"\"Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.\"\"\"\n",
    "\n",
    "articles = list()\n",
    "for link in _test_ls:\n",
    "    _data = article_content(link)\n",
    "            \n",
    "    if _data is not None:\n",
    "\n",
    "        article = {}\n",
    "\n",
    "        article[\"title\"] = \"Sample Title\"\n",
    "        article[\"summary\"] = lorem\n",
    "\n",
    "        _data = article_content(link)\n",
    "\n",
    "        field, _content, img_link = _data\n",
    "\n",
    "        article[\"field\"] = field\n",
    "        article[\"content\"] = _content\n",
    "        article[\"img_link\"] = img_link\n",
    "        \n",
    "        articles.append(article)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "id": "faddfa6e",
   "metadata": {},
   "outputs": [],
   "source": [
    "from jinja2 import Environment, FileSystemLoader"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "id": "5bf6bf75",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[{'title': 'Sample Title',\n",
       "  'summary': \"Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.\",\n",
       "  'field': 'Publishing',\n",
       "  'content': '\\nLast October, the pioneering life-sciences journal eLife introduced bold changes to its editorial practice — which some researchers applauded as reimagining the purpose of a scientific journal. From 31 January this year, eLife said, it would publish every paper it sent out for peer review: authors would never again receive a rejection after a negative review. Instead, reviewers’ reports would be published alongside the paper, together with a short editorial assessment of the work’s significance and rigour. Authors could then decide whether to revise their paper to address any comments.The change followed an earlier decision by eLife to require that all submissions be posted as preprints online. The cumulative effect was to turn eLife into a producer of public reviews and assessments about online research. It was “relinquishing the traditional journal role of gatekeeper”, editor-in-chief Michael Eisen explained in a press release, and “promoting the evaluation of scientists based on what, rather than where, they publish”.\\n\\n\\n\\n\\n\\nThe transformation sparked enthusiastic praise — and sharp criticism. Some scientists saw it as a long-overdue move to empower authors. Others, including some of eLife’s academic editors (who are mostly senior researchers), weren’t so happy. They worried it would diminish the prestige of a brand they’d worked hard to build, and some wrote privately to Eisen (in letters seen by Nature) to say they would resign if the plan was fully implemented. Amid the pushback, the journal postponed switching fully to its new process.But the dispute only heightened. On 9 March, 29 eLife editors — including the journal’s former editor-in-chief, Randy Schekman — wrote to Damian Pattinson, executive director of the journal’s non-profit publisher, eLife Sciences Publications in Cambridge, UK, asking that Eisen be replaced “immediately”. They added that they had no confidence in Eisen’s leadership, because he had dismissed their concerns and had not considered compromise positions. One of the journal’s five deputy editors had already stepped down from that leadership position, and “significant numbers” of reviewers and senior editors were “standing ready to resign”, they wrote.Eisen, a Howard Hughes Medical Institute (HHMI) investigator who works at the University of California, Berkeley, fired back publicly online, tweeting on 12 March that academics were “lobbying hard to get me fired”. He later deleted the tweet, but told Nature in an interview that “opposition to eLife’s model is driven fundamentally by powerful scientists not wanting to change a system that has benefited them and which they have sculpted to continue to reward them”. In response, Schekman and other authors stated that Eisen’s comments were “not true and do not reflect our legitimate concerns with the new model at eLife”.Eisen says he thinks the dissent is small in scale. He and Pattinson say they did not dismiss concerns, but consulted on changes over two years with editors. “We see big swathes of enthusiasm among the community,” Pattinson adds.The row highlights disagreements among researchers about the function of journals and peer review — and, potentially, about the future of science publishing. Some eLife editors argue that journals should use review to guide filtering and rejection of papers. But supporters of eLife’s changes see benefit in preventing peer review from serving as a prestige-gathering function, in which, by rejecting most of the manuscripts submitted to them, selective journals become perceived as arbiters of what work matters. “We rely too much on journal titles in judging people’s work,” Eisen says. “If we want to fix a bad system, we do have to break some eggs.”What is a journal’s purpose?When eLife was launched in 2012 with the financial backing of three powerful science funders — the Maryland-based HHMI, the UK Wellcome Trust and Germany’s Max Planck Society — it had the aim of being a non-commercial and academic-edited journal that would rival prestigious titles such as Cell, Nature and Science. (Nature’s news team is editorially independent of the journal’s research team.) Besides being open access, another of its key innovations was a collaborative system of peer review, where referees and a handling editor discuss comments together. The journal attracted dozens of working scientists as editors who triage submissions, with hundreds more scientists as reviewing editors.\\n\\nOpen-access journal eLife announces ‘preprint first’ publishing model\\neLife had its eye on bigger changes, however. In 2021, the journal decided to publish only papers that were already preprints. This meant that delays in reviewing wouldn’t hold up an author from sharing their work. But even before Eisen and Pattinson joined, the journal had run a trial with more than 300 manuscripts to test the idea of ditching rejection after review. Its aim was to simply publish papers with reviews, author responses and editorial ratings. “The peer-review process does not need to end with a binary outcome of acceptance or rejection,” the journal wrote in a 2019 analysis of that work.It was this idea that eLife instituted for all papers last October, with the addition that editors would also append a short summary assessment of the paper — giving readers a quick idea of its quality and significance. “This puts power back in the hands of the authors, who can then publish what they have, instead of having to do ever more experiments to satisfy reviewers,” says Eisen. The journal plans to charge US$2,000 for the process of arranging review on submissions; previously, its open-access publication fee was $3,000.Some eLife editors are fully on board with the new system. “It’s the future, where science is going,” says senior eLife editor Panayiota Poirazi, a neuroscientist at the Institute of Molecular Biology and Biotechnology in Heraklion, Crete. Among the journal’s funders, HHMI says it fully supports the new policy. Wellcome says that it supports eLife’s publishing process, and the Max Planck Society told Nature it was still discussing the issue.But other researchers have been openly critical from the start. In November, 47 editors wrote privately to Eisen asking for a rethink or for more time to experiment — perhaps running the new system alongside the conventional one, or creating a second journal in which to publish papers of less significance. They worried about harm to the journal’s collaborative open-reviewing process, and that the quality of papers on the eLife platform would drop. With no possibility of rejection, some authors might choose to ignore reviewer comments or only superficially address them, they wrote — and that knowledge might discourage reviewers from producing detailed critiques. Responding to these concerns, Eisen and Pattinson say that they haven’t seen such problems so far, although the project is in its early days, and that running two systems would reduce the chances of the new model’s success.Editors also argued that removing rejection-after-review meant more pressure on the gatekeeping step that remains in eLife’s system — the triage point where editors choose whether to send out a paper for review. That step had been “opaque and subject to errors in judgment”, their letter stated, an issue that would become more consequential if later negative reviews could no longer lead to rejection. Editors might react by becoming more conservative and decide not to take a chance on manuscripts from less-well-known authors. But Eisen says that, in the new system, sending a preprint for review shouldn’t communicate anything about its quality or importance: the reviews and editorial assessments do that instead. The guidance that editors should follow when deciding what to send for review is “can you generate high-quality and broadly useful public reviews of this paper?”, he says.\\n\\nWill the pandemic permanently alter scientific publishing?\\nIn some countries, hiring and promotion decisions still rely heavily on journal titles in candidates’ publication lists — something that is unlikely to change quickly, the editors added in their letter. They worried that scientists there would stop sending their manuscripts to eLife. Eisen, however, says that problematic reliance on journal titles will continue until there is an alternative system, such as eLife’s.In a further private letter sent to Eisen in January, 30 editors said they would resign once the new policy was fully implemented.The full scale of the discontent is unclear. Although Eisen and Pattinson say they’ve had broad support, Axel Brunger, a structural biologist at Stanford University in California, who initiated the first letter, says he reached out only to his colleagues in structural biology and neuroscience, and that nearly all agreed to sign up. “The concerns are widespread,” he says.One researcher who signed all three letters is neuroscientist Gary Westbrook at the Vollum Institute at Oregon Health & Science University in Portland. He is a vocal critic of what he sees as the monopoly that commercial journals have in science publishing, and says he signed “because I didn’t think the new policy was realistic”. Far from helping eLife as a non-profit, high-quality alternative, he says, he thinks the model will diminish its impact.Reviewing preprintsThe concept of reviewing preprints is catching on in the life sciences. At least two dozen preprint-refereeing initiatives of various sizes have been launched in the past few years. The largest (apart from eLife itself) is Review Commons, launched in December 2019 by the California-based non-profit organization ASAPbio and EMBO Press. The latter runs five journals and is part of the European Molecular Biology Organization in Heidelberg, Germany. As a review-sharing collaboration between 17 journals from 6 publishers, including eLife, Review Commons uses EMBO Press editors to select referees for submissions. Authors can ask Review Commons to post reviews and any further author responses on a preprint server, or they can submit their paper, with reviews and responses, to any journal. More than 2,000 reviews of 540 articles have been run through this system.The idea of ‘journal agnostic’ reviewing is still at proof-of-principle stage, says Bernd Pulverer, EMBO’s head of scientific publications. But he sees merit in having both peer-reviewed preprints and conventional journals, which, he says, provide “real added value in condensing and stratifying information”.That view is shared by Maria Leptin, president of the European Research Council. “If I want to learn about a new field that is not core to my own, then I want a trustworthy source that filters for general interest,” she says. “eLife now does its filtering upstream, in a non-transparent, unaccountable way.”The triage stage shouldn’t be seen as this kind of filter, says Eisen. “People are used to operating in a world where appearance in a journal tells you about the quality, audience or import of a study. This is precisely what we are trying to change,” he says. He argues that the short editorial summary eLife appends to its articles serve as quality guides for readers. They grade the significance of the findings (landmark, fundamental, important, valuable, useful) and assess the strength of their support (exceptional, compelling, convincing, solid, incomplete, inadequate).More consultation?Endocrinologist Mone Zaidi at Icahn School of Medicine at Mount Sinai, New York City, is one of eLife’s four remaining deputy editors and has been trying to mediate the issue. He admires Eisen’s vision, he says, “but any new, transformative change has to be done in a careful manner, with buy-in from the community”.Together with some of his colleagues, he is trying to persuade Eisen to slow down, to avoid mass resignations and to establish milestones to assess the effects the changes would have on the lives of working scientists. “There has to be consultation and risk-mitigation plans,” he says.The deputy editor who stood down, cell biologist Anna Akhmanova at the University of Utrecht in the Netherlands, shares Zaidi’s view. She says she helped to develop the new system, but stepped down as deputy editor because it was being pushed through too fast. “We need evolution, not revolution — many small, careful steps to try to move the community towards what would be a better publishing system,” she says.Eisen says he has already responded to concerns by extending — for a short time — the deadline for the regular reviewing system. “We expect things to evolve in interesting ways as people start to see the advantages and opportunities of not making publishing decisions.”“eLife is doing a big and interesting experiment, however it works out,” says stem-cell biologist Fiona Watt, a former eLife deputy editor who is now EMBO’s director. “My sense as a scientist is that the publishing landscape is changing again.”\\n',\n",
       "  'img_link': 'https://media.nature.com/lw767/magazine-assets/d41586-023-00831-6/d41586-023-00831-6_24636118.jpg'},\n",
       " {'title': 'Sample Title',\n",
       "  'summary': \"Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.\",\n",
       "  'field': 'Molecular biology',\n",
       "  'content': '\\n\\n\\n\\n\\n\\n\\nIt is thanks to proteins in the nose called odour receptors that we find the smell of roses pleasant and that of rotting food foul. But little is known about how these receptors detect molecules and translate them into scents.Now, for the first time, researchers have mapped the precise 3D structure of a human odour receptor, taking a step forwards in understanding the most enigmatic of our senses.The study, published in Nature on 15 March1, describes an olfactory receptor called OR51E2 and shows how it ‘recognizes’ the smell of cheese through specific molecular interactions that switch the receptor on.“It’s basically our first picture of any odour molecule interacting with one of our odour receptors,” says study co-author Aashish Manglik, a pharmaceutical chemist at the University of California, San Francisco.Smell mysteryThe human genome contains genes encoding 400 olfactory receptors that can detect many odours. Mammalian odour-receptor genes were first discovered in rats by molecular biologist Richard Axel and biologist Linda Buck in 19912. Researchers in the 1920s estimated that the human nose could discern around 10,000 smells3, but a 2014 study suggests that we can distinguish more than one trillion scents4.Each olfactory receptor can interact with only a subset of smelly molecules called odorants — and a single odorant can activate multiple receptors. It is “like hitting a chord on a piano”, says Manglik. “Instead of hitting a single note, it’s a combination of keys that are hit that gives rise to the perception of a distinct odour.”Beyond this, little is known about exactly how olfactory receptors recognize specific odorants and encode different smells in the brain.Technical challenges in producing mammalian olfactory-receptor proteins using standard laboratory methods have made it difficult to study how these receptors bind to odorants.“Almost all of them really don’t like being in any other kind of cell other than an olfactory sensory neuron,” says Matthew Grubb, a neuroscientist at King’s College London. This means that they cannot be grown or stabilized in commonly used cell lines. “You would have to dissect probably thousands of mice noses” to replicate samples, says Grubb. “It’s just not feasible.”To overcome this, Manglik and his colleagues focused on the OR51E2 receptor, which has functions beyond odorant recognition and is found in gut, kidney and prostate tissues, as well as olfactory neurons.Vinegar and cheeseOR51E2 interacts with two odorant molecules: acetate, which smells like vinegar, and propionate, which has a cheesy odour.The authors purified the receptor and analysed the structure of propionate-bound and unbound OR51E2 using cryo-electron microscopy, an atomic-resolution imaging technique. They also used computer-aided simulations to model how the protein interacts with the odorant at an atomic scale.They found that propionate binds OR51E2 through specific ionic and hydrogen bonds that anchor the propionate’s carboxylic acid to an amino acid, arginine, in a region of the receptor called the binding pocket. Binding to propionate alters the shape of OR51E2, which is what turns the receptor on.These molecular interactions are crucial: the researchers showed that mutations affecting arginine prevented OR51E2 from being activated by propionate.“This is our way of kind of lining up the dominoes to understand how pushing on one side of the receptor turns the other side on,” says Manglik.On the scentScientists have long dreamed of building a molecular atlas of olfactory receptors that maps their chemical structures and which combinations of receptors correspond to particular odours. But “that’s been very much out of reach for the field”, says Manglik.The OR51E2 receptor is specific to propionate and acetate. But “it’s not all about single odorant binding to single receptor molecules”, says Grubb. OR51E2 is a class I olfactory receptor; only around 10% of human olfactory-receptor genes encode this type. The rest code for class II receptors, which typically recognize a broader range of odours. “They may have very different mechanisms,” says Vanessa Ruta, a neuroscientist at the Rockefeller University in New York City.Studying other examples of human odour receptors and elucidating their structures is crucial, she adds. “It will allow for a broader understanding of the different ways that odorants are recognized.”\\n',\n",
       "  'img_link': 'https://media.nature.com/lw767/magazine-assets/d41586-023-00818-3/d41586-023-00818-3_24631076.jpg'},\n",
       " {'title': 'Sample Title',\n",
       "  'summary': \"Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.\",\n",
       "  'field': 'Computer science',\n",
       "  'content': \"\\n\\n\\n\\n\\n\\n\\nArtificial intelligence company OpenAI this week unveiled GPT-4, the latest incarnation of the large language model that powers its popular chat bot ChatGPT. The company says GPT-4 contains big improvements — it has already stunned people with its ability to create human-like text and generate images and computer code from almost any a prompt. Researchers say these abilities have the potential to transform science — but some are frustrated that they cannot yet access the technology, its underlying code or information on how it was trained. That raises concern about the technology’s safety and makes it less useful for research, say scientists.One upgrade to GPT-4, released on 14 March, is that it can now handle images as well as text. And as a demonstration of its language prowess, Open AI, which is based in San Francisco, California, says that it passed the US bar legal exam with results in the ninetieth centile, compared with the tenth centile for the previous version of ChatGPT. But the tech is not yet widely accessible — only to paid subscribers to ChatGPT so far have access.\\n\\nChatGPT listed as author on research papers: many scientists disapprove\\n“There’s a waiting list at the moment so you cannot use it right now,” Says Evi-Anne van Dis, a psychologist at the University of Amsterdam. But she has seen demos of GPT-4. “We watched some videos in which they demonstrated capacities and it’s mind blowing,” she says. One instance, she recounts, was a hand-drawn doodle of a website, which GPT-4 used to produce the computer code needed to build that website, as a demonstration of the ability to handle images as inputs.But there is frustration in the science community over OpenAI’s secrecy around how and what data the model was trained, and how it actually works. “All of these closed-source models, they are essentially dead-ends in science,” says Sasha Luccioni, a research scientist specializing in climate at HuggingFace, an open-source-AI community. “They [OpenAI] can keep building upon their research, but for the community at large, it’s a dead end.”‘Red team’ testingAndrew White, a chemical engineer at University of Rochester, has had privileged access to GPT-4 as a ‘red-teamer’: a person paid by OpenAI to test the platform to try and make it do something bad. He has had access to GPT-4 for the past six months, he says. “Early on in the process, it didn’t seem that different,” compared with previous iterations.\\n\\nAbstracts written by ChatGPT fool scientists\\nHe put to the bot queries about what chemical reactions steps were needed to make a compound, predict the reaction yield, and choose a catalyst. “At first, I was actually not that impressed,” White says. “It was really surprising because it would look so realistic, but it would hallucinate an atom here. It would skip a step there,” he adds. But when as part of his red-team work he gave GPT-4 access to scientific papers, things changed dramatically. “It made us realize that these models maybe aren’t so great just alone. But when you start connecting them to the Internet to tools like a retrosynthesis planner, or a calculator, all of a sudden, new kinds of abilities emerge.”And with those abilities come concerns. For instance, could GPT-4 allow dangerous chemicals to be made? With input from people such as White, OpenAI engineers fed back into their model to discourage GPT-4 from creating dangerous, illegal or damaging content, White says.Fake factsOutputting false information is another problem. Luccioni says that models like GPT-4, which exist to predict the next word in a sentence, can’t be cured of coming up with fake facts — known as hallucinating. “You can’t rely on these kinds of models because there’s so much hallucination,” she says. And this remains a concern in the latest version, she says, although OpenAI says that it has improved safety in GPT-4.Without access to the data used for training, OpenAI’s assurances about safety fall short for Luccioni. “You don’t know what the data is. So you can’t improve it. I mean, it’s just completely impossible to do science with a model like this,” she says.\\n\\nHow Nature readers are using ChatGPT\\nThe mystery about how GPT-4 was trained is also a concern for van Dis’s colleague at Amsterdam, psychologist Claudi Bockting. “It’s very hard as a human being to be accountable for something that you cannot oversee,” she says. “One of the concerns is they could be far more biased than for instance, the bias that human beings have by themselves.” Without being able to access the code behind GPT-4 it is impossible to see where the bias might have originated, or to remedy it, Luccioni explains.Ethics discussionsBockting and van Dis are also concerned that increasingly these AI systems are owned by big tech companies. They want to make sure the technology is properly tested and verified by scientists. “This is also an opportunity because collaboration with big tech can of course, speed up processes,” she adds.Van Dis, Bockting and colleagues argued earlier this year for an urgent need to develop a set of ‘living’ guidelines to govern how AI and tools such as GPT-4 are used and developed. They are concerned that any legislation around AI technologies will struggle to keep up with the pace of development. Bockting and van Dis have convened an invitational summit at the University of Amsterdam on 11 April to discuss these concerns, with representatives from organizations including UNESCO’s science-ethics committee, Organisation for Economic Co-operation and Development and the World Economic Forum.Despite the concern, GPT-4 and its future iterations will shake up science, says White. “I think it's actually going to be a huge infrastructure change in science, almost like the internet was a big change,” he says. It won’t replace scientists, he adds, but could help with some tasks. “I think we're going to start realizing we can connect papers, data programmes, libraries that we use and computational work or even robotic experiments.”\\n\",\n",
       "  'img_link': 'https://media.nature.com/lw767/magazine-assets/d41586-023-00816-5/d41586-023-00816-5_24630796.jpg'}]"
      ]
     },
     "execution_count": 30,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "articles"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 53,
   "id": "af6b2246",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "... wrote sample.html\n"
     ]
    }
   ],
   "source": [
    "results_filename = \"sample.html\"\n",
    "results_template = environment.get_template(\"index.html\")\n",
    "context = {\n",
    "    \"datetime\" : tuple( ((datetime.today() - timedelta(weeks=1)).strftime(\"%d/%m/%Y\"), datetime.today().strftime(\"%d/%m/%Y\")))\n",
    "    \"data\": data,\n",
    "}\n",
    "with open(results_filename, mode=\"w\", encoding=\"utf-8\") as results:\n",
    "    results.write(results_template.render(context))\n",
    "    print(f\"... wrote {results_filename}\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "id": "18739eaa",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'14/03/2023'"
      ]
     },
     "execution_count": 8,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "datetime.today().strftime(\"%d/%m/%Y\")"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "news",
   "language": "python",
   "name": "news"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.8.16"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
